\chapter{Introduction}

\section{The evolution of gene regulation}

The study of human genetics seeks to understand the genetic basis behind human phenotypic variation. Specifically, human geneticists are interested in characterizing how genetics impacts differences in traits between species, as well as between individuals within our own species. Insights gleaned from this field of study are useful not only for understanding and potentially improving human health, but also for furthering our understanding of biology and evolution more broadly. One particularly compelling approach that addresses both these aims is to compare genetic and phenotypic variation between humans and closely-related primate species. Early comparisons between the human and chimpanzee genomes revealed that the two species share the vast majority (99\%) of their DNA, with an especially high level of conservation in protein-coding sequences \cite{King&Wilson + "great ape genetic diversity and population history" + "Initial sequence of the chimpanzee genome and comparison with the human genome"}. These observations provided amazing evolutionary insight, suggesting that differences observed between humans and chimpanzees are not driven by inherent differences in the proteins (genes), but, rather, by differences in how, when, and where these proteins are expressed--i.e. differences in gene regulation \cite{King&Wilson}. More recently, genome-wide association studies (GWAS) within the human species have connected inter-individual differences in traits and diseases with thousands of genetic variants, the vast majority of which are non-coding \cite{Edwards et al 2013 (Beyond GWASs: Illuminating the Dark Road from Association to Function), Gloss and Dinger 2018 (Realizing the significance of noncoding functionality in clinical genomics)}. Additionally, numerous estimates suggest the human genome is comprised of up to 98\% noncoding DNA, much of which is functional, \cite{Pennacchio et al. 2013 (Enhancers: 5 essential ?s), Comfort 2015 (Genetics: We are the 98%), Kellis et al. 2014 (Defining functional DNA elements in the human genome), Consortium 2001 (Initial sequencing and analysis of the human genome), Little 2005 (Structure and function of the human genome)}, further underscoring the importance of understanding gene regulation.

The hypothesis that gene regulatory differences may be major drivers of phenotypic variation was first posited more than 50 years ago \cite{Britten and Davidson 1969+1971}. Although there is still some debate about the extent to which adaptation and speciation are driven by mutations in gene regulatory loci vs. mutations in protein-coding genes \cite{Hoekstra and Coyne 2007, Sean B Carroll 2008}, it is evident that gene expression variance plays a crucial role in phenotypic divergence within and between species \cite{Sean B Carroll 2005, Greg Wray 2007 (evolutionary significance of cis-regulatory mutations), Gilad et al. 2006 (expression profiling in primates...), Frankel Wang and Stern 2012 (Conserved regulatory architecture...), Stern and Orgogozo 2008 (The loci of evolution...), Zheng et al. 2011 (Regulatory variation within and between species)}. Advances in molecular biology and sequencing technologies over the last several decades have enabled rigorous investigations of gene expression levels and the regulatory mechanisms affecting them. RNA-sequencing (RNA-seq) represents a major improvement over prior microarray technology for accurate genome-wide measurement of gene expression levels and other transcriptomic phenotypes  \cite{Zhao et al 2014 (Comparison of RNA-seq and Microarray...activated T cells), Mantione et al 2014 (Comparing Bioinformatic gene expression...), Wang et al. 2009 (RNA-seq: a revolutionary tool for transcriptomics), Marioni et al 2008 (RNA-seq: an assessment of technical reproducibility and comparison with gene expression arrays)}. Simultaneously, an array of emerging molecular and computational techniques allow for genome-wide assessment of different regulatory mechanisms such as DNA methylation, histone modification, chromatin state, transcription factor binding, and more \cite{Chen et al. 2017 (Epigenetic Regulation: A new frontier for biomedical engineers)}. Widespread use of these techniques is needed, because, in contrast to protein-coding genes, there is no clear code connecting primary sequence to downstream function for the noncoding portions of the genome. The outstanding challenges are thus to characterize the different genetic and epigenetic mechanisms regulating gene expression, to understand their relative evolutionary contributions to adaptation and speciation, and to ultimately connect them back to variation in primary DNA sequence.

While my work and this introduction focuses on inter-species primate comparative genomics in order to understand the evolution of gene regulation, it should be noted that research on gene expression differences within species has yielded valuable insights, suggesting expression levels are heritable and connected to genetic variation through expression quantitative trait loci (eQTLs) \cite{Majewski and Pastinen 2010 (The study of eQTL variations...), Gilad, Rifkin, and Pritchard 2008 (Revealing the architecture...)}. These findings further highlight the functional relevance of gene expression levels as a molecular phenotype affecting higher-order traits, providing a tractable avenue for understanding the evolution of a wide variety of phenotypes. In the rest of this introduction, I will review key findings from comparative primate genomics studies about the evolution of gene regulation, and explain why characterizing the 3-dimensional structure of the genome is a critical next step in understanding the evolution of gene regulation and the relationships between genotypes and phenotypes.

\section{Gene regulatory evolution insights from comparative primate genomics}

\textit{Why focus on primates?} Numerous efforts have made progress in decoding the non-coding genome and furthering our understanding of gene regulation. In particular, work from the ENCODE (Encyclopedia of DNA Elements) consortia and many others have helped to identify and characterize functional regulatory elements across a number of species, including humans \cite{Bernstein et al 2010 (The NIH Roadmap Epigenomics Mapping Consortium), ENCODE Project Consortium 2012 (An integrated...in the human genome), Mouse ENCODE Consortium 2012 (An encyclopedia of mouse DNA elements), Celniker et al. 2009 (Unlocking the secrets of the genome)}. Inter-species comparative genomics studies have been especially effective in identifying functional regulatory sequences \cite{Nobrega and Pennacchio 2004 (Comparative genomic analysis as a tool for biological discovery)}. In addition to characterizing specific regulatory loci, studies in model organisms have also revealed that expression divergence between species is primarily driven by mutations in cis-regulatory elements (CREs), rather than trans elements \cite{Tirosh and Barkai 2011 (Inferring regulatory mechanisms...)}. The former operate in an allele-specific manner, typically on the same chromosome, while the latter operate more broadly and can often diffuse throughout the genome. Although these research endeavors have expanded knowledge on gene regulatory mechanisms and associated loci more broadly, their use of distantly-related species precludes the possibility of categorizing their findings as highly conserved or human-specific \cite{Prime time for Primate Functional Genomics}. Grounding genetic and epigenetic observations in humans by comparison with closely-related non-human primates (NHP) is crucial for providing evolutionary context. Without such comparisons, it is impossible to obtain a comprehensive understanding of how different loci and mechanisms of gene regulation have evolved in the human lineage, and, consequently, how these features may affect human-specific phenotypes \cite{Romero, Ruvinsky, and Gilad 2012}. In addition to providing evolutionary context, using NHP in comparative genomics and biomedical research can yield useful insights into human diseases, which are harder to model in more distantly related species with more divergent physiologies \cite{Rogers and Gibbs 2014 (Comparative primate genomics: emerging patterns of genome content and dynamics)}. There are thus a number of different ways in which comparative genomics studies utilizing NHP may provide unique insights into human evolution and the lexicon of human gene regulation, that cannot be obtained by focusing exclusively on model organisms and/or more distantly related species.

However, using primate comparative genomics to understand gene regulation still entails a number of challenges. The paucity of human and chimpanzee primary tissues, as well as obvious ethical limitations on experimentation in the two species, represent major barriers in the study of gene regulation \cite{Romero, Ruvinsky, and Gilad 2012}. Comparison of biological samples between primate species is possible with post-mortem collection of flash-frozen tissues, but this approach is problematic for several reasons. Due to the inherently opportunistic sample collection, sample sizes are typically quite small, with some studies using only a few individuals from each species \cite{Blekhman et al 2008 (Gene regulation in primates evolves under...), Pai et al 2011 (A genome-wide study of DNA methylation), Prescott et al 2015 (Enhancer Divergence...Chimp Neural Crest)}. Additionally, post-mortem tissue samples may be subject to variance induced by technical factors such as sample collection and shipping \cite{Blake et al. 2020 (A comparison of gene expression and DNA methylation), Chevyreva et al. 2007 (Assessing RNA quality in postmortem human brain tissue)}. These issues may be mitigated by utilizing induced pluripotent stem cells (iPSCs), which can be reprogrammed from and differentiated into a wide variety of different cell types \cite{Takahashi & Yamanaka 2006, Takahashi et al. 2007 (Induction of pluripotent stem cells...), Sun et al 2009 (Feeder-free derivation...))}, allowing for controlled experimentation on larger panels of human and NHP cells \cite{Romero et al 2015 (A panel of induced pluripotent stem cells from chimpanzees...), Marchetto et al. 2013 (Differential L1 regulation...)}. Regardless of the biological samples being compared, care must also be taken in the study design and analysis methods employed in a comparative genomics setting. Without careful study design, batch effects may have a strong impact on the data, leading to inferences and conclusions that are driven more by technical variables than by true biological differences. In one intriguing recent example, researchers observed gene expression data from human and mouse clustering by species, rather than by tissue (as would be expected based on prior research) \cite{Mouse ENCODE Consoritum, An Encyclopedia of DNA Elements in the Mouse Genome}. A reanalysis of the data found that this unexpected observation was due to flawed study design confounding sequencing batch with species, and that the data do indeed cluster by tissue after accounting for sequencing batch effects \cite{Gilad and Mizrahi-Man 2015}. With respect to the analysis of comparative genomics data, thoughtful normalization and orthology-calling techniques must also be employed to ensure comparisons made between sequences and features lead to valid biological inferences \cite{Vallender 2009 (Bioinformatics approaches to identifying orthologs and assessing evolutionary relationships), Zhou et al. 2019 (A statistical normalization method and differential expression analysis for RNA-seq data between different species), Blekhman et al. 2010 (Sex-specific and lineage-specific alternative splicing in primates), Cain et al. 2011 (Gene expression differences among primates are associated with changes in a histone epigenetic modification)}. Intentional study designs, sample collection methods, and analytical techniques can attenuate many of the aforementioned limitations.

Even when researchers do well to address technical limitations and minimize confounding variables, there are still challenges in inferring regulatory evolutionary dynamics from comparative primate genomics studies. Numerous models and methodologies exist to infer the action of natural selection on primary DNA sequence \cite{Vitti Grossman and Sabeti 2013, Detecting Natural Selection in Genomic Data}. Such methods are particularly effective when applied to protein-coding genes, where the functional effect of a mutation can be understood readily, given our knowledge of the link between sequence codons and the amino acids they are translated into. However, the links between primary sequence and trait variation are considerably less clear in the context of epigenetic, regulatory, and other molecular phenotypes. In these cases, the action of natural selection on the trait in question can be inferred and statistically tested based on observed deviations from null models (e.g. a neutral model with no selection). These types of approaches can be extremely useful in model organisms, where one can directly measure the necessary parameters (e.g. mutation rate) to generate a reasonable null model \cite{Gilad Oshlack and Rifkin 2006, Natural selection on gene expression}. Unfortunately, these parameters are difficult to measure even in model organisms, and can often be practically impossible to estimate in humans and NHP. For most functional genomic traits that are studied, there is not yet a robust, well-formulated null model of no selection. Thus, more ad-hoc and empirical approaches must be utilized to understand the action of natural selection on intermediate molecular phenotypes. For instance, if gene expression levels show low variation both within and between species, it may be inferred that regulation of these genes is evolving under stabilizing selection (i.e. expression extremes are selected against). Conversely, if expression variance is low within species, but mean expression is much higher or lower in one species compared to others, this may suggest directional selection is affecting regulation of the gene in that species \cite{Romero, Ruvinsky, and Gilad 2012}. Empirical approaches such as these can therefore elucidate evolutionary dynamics, but greater care must be taken in interpretation of their results. In the previous example, more extreme expression in one species may actually be due to environmental factors and/or reduced action of stabilizing selection on that lineage, rather than directional selection. Functional follow-up studies, consideration of interspecies environmental differences, and accounting for different possible evolutionary trajectories can help exclude alternative explanations for specific evolutionary inferences. Consequently, when executed and interpreted properly, comparative primate genomics studies have vastly expanded our understanding of the mechanisms and loci involved in the evolution of gene regulation.

\textit{What have we learned?}
For many genes, results from comparative studies suggest that expression levels in primates are evolving under natural selection \cite{Khaitovich et al 2006 (Evolution of primate gene expression), Blekhman et al 2008 (Gene regulation in primates evolves under tissue-specific selection pressures), Brawand et al 2011 (The evolution of gene expression levels in mammalian organs)}. One early study looked at RNA levels in liver tissues from humans, chimpanzees, orangutans, and rhesus macaques, finding a set of genes with relatively invariant expression levels across species \cite{Gilad et al 2006 (Expression profiling in primates...)}. If regulatory mutations are in general selectively neutral (as many other mutations have been speculated to be \cite{Kimura 1968 (Evolutionary rate at the molecular level)}), expression levels for most genes would show more substantial inter-primate variation. The observed low variance in expression across diverse primate lineages suggests stabilizing selection has acted on these genes, preventing extreme expression levels \cite{Lemos et al 2007, Rates of divergence in gene expression profiles of...}. Other studies comparing RNA levels across primate species have largely corroborated this notion \cite{Blekhman et al 2008, Perry et al. 2011 (Comparative RNA sequencing reveals...), Brawand et al 2011, Chen et al. 2018 (A quantitative framework for characterizing the evolutionary history of mammalian gene expression)}. It is interesting to note that interspecies conservation of expression levels is particularly high for genes thought to be critical for defining cell type identity. In turn, gene expression patterns are more similar in the same tissue across different species than across different tissues within a species \cite{Sudmant et al. 2015 (Meta-analysis of RNA-seq expression data...)}. Taken together, these results bolster the idea that regulatory changes may be crucial drivers of evolution and adaptation. Compared to protein-coding sequence mutations, regulatory mutations can act in a more tissue-specific manner, and are thus less susceptible to pleiotropic effects that could be deleterious across multiple organs \cite{Wilson et al. 1974 (Two types of molecular evolution...), Blekhman et al 2008}.

While most genes show evidence for their expression evolving under stabilizing selection in primates, there has also been great interest in finding examples of directional selection. The impact of such examples is fairly intuitive: a strong motivation in comparative primate genomics is to identify the genetic and epigenetic facets underlying differences between humans and NHP, in an effort to understand the biology behind human-specific traits and diseases. The exact proportion of genes whose regulation appears to be evolving under directional selection in primates differs dependent upon the tissue or cell type being considered, but still represents the minority of genes \cite{Romero, Ruvinsky and Gilad 2012}. As discussed above, not every gene showing lineage-specific expression differences compared to other primates is necessarily evolving under positive (directional) selection on its regulation, but many likely are. Since some of the most striking phenotypic differences between humans and NHP are cognitive, many primate comparative genomics studies have focused on the brain and specific cell types therein \cite{Nowick et al 2009 (Difference in human and chimpanzee gene expression patterns define an evolving network of transcription factors in brain), Giger et al. 2010 (Evolution of Neuronal and Endothelial Transcriptomes in Primates)}. Results from studies utilizing bulk RNA-seq \cite{Somel et al 2009 (Transcriptional neoteny in the human brain)}, and from more recent work examining RNA transcripts in single cells \cite{Mora-Bermudez et al. 2016 (Differences and similarities between human and chimpanzee neural progenitors...), Zhu et al. 2018 (Spatiotemporal transcriptomic divergence across human and macaque brain development), Sousa et al. 2017 (Molecular and cellular reorganization of neural circuits in the human lineage)}, suggest that interspecies expression variation in tissue location (heterotopy) and timing (heterochrony) during brain development may play a role in cognitive differences observed between humans and NHP. There are also examples in other organs of directional selection acting on gene regulation. An RNA-seq study on livers from 16 species revealed expression changes in some primate lineages that could be tied to dietary adaptations \cite{Perry et al 2011 (Comparative RNA sequencing reveals substantial...)}. A similar study examining livers, kidneys, and hearts in humans, chimpanzees, and rhesus macaques found subsets of genes in each tissue displaying lineage-specific expression in humans \cite{Blekhman et al 2008}. Yet another study found marked expression differences in blood leukocytes, livers, and brains of humans, chimpanzees, orangutans, and rhesus macaques \cite{Enard et al 2002 (Intra- and Interspecific Variation in Primate Gene Expression Patterns)}. A more recent study assayed RNA levels in heart, kidney, liver, and lung tissue samples from humans, chimpanzees, and rhesus macaques, and also found a minority of genes whose expression patterns imply some directional selection \cite{Blake et al. 2020 (A comparison of gene expression and DNA methylation patterns...)}. Specific and functionally validated examples of genes with expression levels evolving under directional selection are scarce, and it is difficult to confidently connect them to higher-order phenotypes \cite{Babbitt et al 2010 (Multiple functional variantsin cis modulate PDYN expression), Warner et al. 2009 (Functional consequences of genetic variation in primates on tyrosine hydroxylase (TH) expression in vivo)}. More examples will hopefully be discovered and characterized as our understanding of gene-phenotype connections increases, and as single-cell and other technologies enable better sampling from different locations, time points, and, consequently, cell types within a tissue. Regardless, numerous lines of evidence from inferences about directional selection suggest that a complex network of different mechanisms regulates gene expression. Gene sets with some evidence for directional selection on their regulation are often enriched for transcription factors \cite{Blekhman et al 2008, Gilad et al. 2006 (Expression profiling in primates revels a rapid evolution of human transcription factors)}, are regulated by fewer enhancers than genes with more conserved expression patterns \cite{Danko et al 2018 (Dynamic evolution of regulatory element ensembles in primate CD4+ T cells), Berthelot et al. 2018 (Complexity and conservation of regulatory landscapes underlie evolutionary resilience of...)}, and do not necessarily display signatures of directional selection in the abundance of their corresponding proteins \cite{Khan et al 2013 (Primate transcript and protein expression levels evolve under compensatory selection pressures)}. This last observation is particularly intriguing, and suggests that protein levels may be under more selective constraint than RNA levels and other gene regulatory mechanisms \cite{Wang et al 2018 (Post-translational buffering leads to convergent protein expression levels between primates), Anderson Vigalys and Tung 2020 (Broadening primate genomics: new insights into the ecology and evolution of primate gene regulation)}. An emerging notion from this and other work is that regulatory mechanisms utilize such buffering and redundancy to ensure appropriate downstream functional outcomes \cite{MacNeil and Walhout 2011 (Gene regulatory networks and the role of robustness and stochasticity in the control of gene expression)}. Together, these findings highlight the importance of measuring a wide variety of epigenetic features in order to understand the evolution of gene regulation.

Indeed, more recent work has moved from merely characterizing expression differences across primate species, to attempting to understand variation in regulatory mechanisms driving these differences. Understanding the regulatory mechanisms and loci responsible for expression variation should help de-mystify the noncoding portions of the genome, identifying functional elements that may have an impact on human health \cite{Cooper and Shendure 2011 (Needles in stacks of needles: finding disease-causal variants in a wealth of genomic data)}. One recent study examined DNA methylation and gene expression in livers, kidneys, hearts, and lungs from humans, chimpanzees, and rhesus macaques, finding that methylation differences can only explain a small proportion of expression variation between tissues and species \cite{Blake et al 2020 (A comparison of gene expression and DNA methylation...)}. An earlier study found similar results when comparing human and chimpanzee livers, hearts, and kidneys \cite{Pai et al 2011 (A Genome-Wide study of DNA methylation...)}. Differential abundance of microRNAs, which regulate mRNA transcript decay, was also observed to account for only a small proportion (<5\%) of expression differences in prefrontal cotex across humans, chimpanzees, and rhesus macaques \cite{Hu et al. 2011 (MicroRNA expression and regulation in human...), Somel et al. 2011 (MicroRNA-driven developmental remodeling in the brain...)}. Alternative splicing of genes, which could (in theory) easily introduce regulatory novelty, has also been shown to have little effect on differential expression between humans and chimpanzees \cite{Calarco et al. 2007 (Global analysis of alternative splicing differences between humans and chimpanzees)}. Comparable observations have been made for histone marks: one early study examined H3K4me3 in lymphoblastoid cell lines (LCLs) from humans, chimpanzees, and rhesus macaques, and found that interspecies differences in this histone modification explain only 7\% of interspecies expression variation \cite{Cain et al 2011 (Gene expression differences among primates are associated with changes in a histone epigenetic modification)}. These results are perhaps not surprising, given that only a small minority of loci show human-specific increase or decrease of H3K4me3 in prefrontal cortex samples from the same species \cite{Shulha et al 2012 (Human-specific histone methylation signatures at transcription start sites in prefrontal neurons)}. A later analysis in LCLs from the same species integrated RNA polymerase II occupancy, H3K4me1, H3k4me3, H3K27ac, and H3K27me3, and found that roughly 40\% of interspecies gene expression variance can be explained by these marks combined \cite{Zhou et al. 2014 (Epigenetic modifications are associated with inter-species gene expression variation in primates)}. Similarly, an even more recent study integrated a wide variety of histone marks as well as chromatin accessibility and methylation status in LCLs from great apes and macaques, and found that these features combined can explain approximately 67\% of interspecies expression variance \cite{Garcia-Perez et al 2020 (Biorxiv, Epigenomic profiling of primate LCLs reveals the coordinated evolution of gene expression and epigenetic signals in regulatory architectures)}. It would thus appear that the effect of any single epigenetic mechanism on gene expression is relatively modest, but, in concert, these mechanisms have a large effect on expression levels.

One principle that emerges from these findings and others in model organisms is that chromatin state and its effects on cis-regulatory elements (CREs) play a major role in the evolution of gene expression \cite{Romero, Ruvinsky and Gilad 2012, Degner et al 2012 (DNase I sensitivity QTLs are...)}. Many of the aforementioned epigenetic modifications and mechanisms are associated with how 'open' or 'closed' chromatin is at a given locus, making the locus and the CREs it encompasses more or less accessible to transcription factors and other regulatory machinery \cite{Calo and Wysocka 2013 (Modification of enhancer chromatin: what, how and why?)}. It is therefore important to assess not only whether a given CRE is conserved, but also if the chromatin state at that locus is comparable across species. A variety of methods exist to assay chromatin accessibility, such as DNase-seq and ATAC-seq \cite{Klein and Hainer 2019 (Genomic methods in profiling DNA accessibility and factor localization), Tsompana and Buck 2014 (Chromatin accessibility: a window into the genome), Buenrostro et al 2015 (ATAC-seq: A method for assaying chromatin accessibility genome-wide)}. Broadly, these methods have found that the vast majority of the genome is not accessible in any given cell type, and that most transcription factor binding events occur within regions of open chromatin identified by these assays \cite{Thurman et al. 2012 (The accessible chromatin landscape of the human genome)}. Consequently, regions of open chromatin are often likely to harbor active regulatory elements \cite{Song et al. 2011 (Open chromatin defined by DNaseI and FAIRE identified regulatory elements that shape cell-type identity), Klemm et al. 2019 (Chromatin accessibility and the regulatory epigenome)}. Consortia such as ENCODE have used ChIP-seq and other techniques to produce copious histone mark and other epigenetic data that, through careful analysis, have helped identify and characterize different classes of CREs at these putative regulatory regions\cite{ENCODE Project Consortium 2012 (An integrated...in the human genome), Li et al. 2018 (Genome-wide prediction of cis-regulatory regions using supervised deep learning methods), Ernst and Kellis 2012 (ChromHMM: automating chromatin-state discovery and characterization), Hoffman et al. 2012 (Unsupervised pattern discovery in human chromatin structure through genomic segmentation)}. Some of the most well-studied CREs are enhancers, DNA modules that interface with transcription factors and associated proteins to make contact with gene promoters, thereby affecting gene expression. Although the measured extent of inter-primate divergence or conservation in enhancers and other CREs is dependent on the technology used and the number of species examined \cite{Edsall et al. 2019 (Evaluating chromatin accessibility differences across multiple primate species using a joint modeling approach), Swain-Lenz et al 2019 (Comparative analyses of chromatin landscape in white adipose tissue suggest humans may have less beigeing...)}, there are intriguing enhancer differences across primate species. Enhancers are considerably less conserved amongst primate lineages than gene expression levels \cite{Villar et al. 2015 (Enhancer evolution across 20 mammalian species), Berthelot et al. 2017 (Complexity and conservation of regulatory landscapes underlie evolutionary resilience of mammalian gene expression)} and less conserved than promoters \cite{Trizzino et al 2017 (Transposable elements are the primary source of novelty in primate gene regulation)}, highlighting their evolutionary relevance. Surveys of enhancers across primate and mammalian evolution have found interspecies differences in their activity \cite{Klein et al. 2018 (Functional characterization of enhancer evolution in the primate lineage), Prescott et al 2015 (Enhancer divergence and cis-regulatory evolution in the human and chimp neural crest), Shibata et al 2012 (Extensive evolutionary changes in regulatory element activity during human origins...)}, and evidence for high evolutionary turnover of enhancers as compared to promoters \cite{Carelli et al 2018 (Repurposing of promoters and enhancers during mammalian evolution), Villar et al. 2015 (Enhancer evolution across 20 mammalian species)}.

While tremendous work has been done to identify and characterize enhancers and other CREs, numerous outstanding questions remain. Identification of enhancers is still imperfect because there is no single epigenetic mark that perfectly predicts enhancer regions. Enhancers affect gene expression via chromatin looping but also through other mechanisms that are less well understood (e.g. enhancer transcription), and disease-associated genetic variation at enhancer regions is difficult to functionally characterize \cite{Enhancers: 5 essential questions}. Although we have been able to map regulatory quantitative trait loci (QTL, genetic variants with discrete effects on expression or other intermediate molecular phenotypes), most disease-associated variants in regulatory regions do not appear as QTL \cite{Umans et al. 2020 (Where are the disease-associated eQTLs?)}. This is likely due in part to sampling strategies (e.g. not assaying gene expression in the appropriate cell type or condition relevant to the disease), but progress in understanding these mutations is also broadly stymied by a lack of knowledge about which gene(s) a given CRE regulates \cite{Umans et al. 2020}. Determining CREs? targets is of particular importance both because CREs act in a distance-independent manner, and because many CREs are tissue-specific in their activity \cite{Levine 2010 (Transcriptional enhancers in animal development and evolution), Ong et al. 2011 (Enhancer function new insights into the regulation of tissue-specific gene expression), Allis et al. 2016 (The molecular hallmarks of epigenetic control), Won et al. 2008 (Prediction of regulatory elements in mammalian genomes using chromatin signatures)}. Ultimately, CREs? tissue-specific effects on gene expression are likely to be principally determined by local chromatin state and by what gene promoter(s) CREs come into contact with \cite{Garcia-Gonzalez et al 2016 (Chromatin remodeling effects on enhancer activity)}. Connecting regulatory elements directly with their targets thus represents a crucial step towards obtaining a complete picture of how CREs modify expression, and how mutations in CREs affect higher-order phenotypes. My thesis work addresses this issue by examining expression divergence between humans and chimpanzees, with a focus on comparative assessment of CRE-gene contacts in 3-dimensional genome space.

\section{The growing importance of the 3D genome}
Given the vast wealth of genomic information that each cell has to carry in its nucleus, eukaryotic genomes must be packaged in a highly complex and structured fashion. At the highest level, individual chromosomes preferentially occupy discrete regions of the nucleus ("chromosome territories"), and these territories are fairly conserved across primate lineages \cite{Meaburn and Misteli 2007 (Chromosome territories), Mora et al. 2006 (Chromosome territory positioning of conserved homologous chromosomes in different primate species), Tanabe et al. 2002 (Evolutionary conservation of chromosome territory arrangements in cell nuclei from higher primates)}. Early studies of 3D genome structure largely relied on imaging techniques such as FISH (fluorescence in-situ hybridization), and did well to uncover chromosome territories; the advent of molecular methodologies like chromosome conformation capture (3C) have enabled interrogation of 3D genome structure at even finer scales \cite{Dekker et al 2002 (Capturing chromosome conformation)}. Since the inception of 3C, the technology has developed through a variety of iterations and improvements that have increased its genomic resolution and throughput \cite{Fraser et al. 2015 (An overview of genome organization and how we got there: from FISH to Hi-C)}. The latest version, known as Hi-C, pairs the original method?s proximity-based ligation with high throughput next-generation sequencing to find DNA-DNA contacts genome-wide \cite{Lieberman-Aiden et al. 2009 (Comprehensive mapping of long-range interactions reveals folding principles of the human genome)}. These techniques have revealed that, beneath the scale of chromosome territories, individual chromosomes are also partitioned into two types of large-scale "compartments": A compartments, representing open and transcriptionally accessible chromatin, and B compartments, representing closed chromatin \cite{Lieberman-Aiden et al. 2009 (Comprehensive mapping of long-range interactions reveals folding principles of the human genome), Naumova and Dekker 2010 (Integrating one-dimensional and three-dimensional maps of genomes)}. At an even finer scale, loci within a compartment appear to form self-interacting regions on the scale of a megabase, termed topologically associating domains (TADs) \cite{Dixon et al. 2012 (Topological domains in mammalian genomes identified by analysis of chromatin interactions), Nora et al. 2012 (Spatial partitioning of the regulatory landscape of the X-inactivation centre), Hou et al. 2012 (Gene density, transcription, and insulators contribute to the partition of the Drosohila genome into physical domains), Sexton et al. 2012 (Three-dimensional folding and functional organization principles of the drosophila genome)}. As I discuss further in the third chapter of this thesis, the definition of a TAD is still changing as Hi-C libraries are more deeply sequenced and new TAD inference algorithms arise. Regardless, loci within a TAD make contact with one another much more frequently than they do with loci outside of the TAD, suggesting that TADs may represented neighborhoods of insulated gene regulation that constrain the possible set of gene-CRE interactions \cite{Andrey and Mundlos 2017 (The three-dimensional genome: regulating gene expression during pluripotency and development), Symmons et al. 2014 (Functional and topological characteristics of mammalian regulatory domains), Sexton and Cavalli 2015 (The role of chromosome domains in shaping the functional genome)}. Lastly and perhaps most importantly, at the lowest scale, Hi-C and related techniques have uncovered individual DNA looping interactions that bring linearly distant CREs into proximity with the genes they regulate \cite{Rao et al. 2014 (A 3D map of the human genome at kilobase resolution...), Kadauke and Blobel 2008 (Chromatin loops in gene regulation)}.

Numerous lines of evidence point to the functional significance of 3D genome organization at different scales. Studies have found that a significant fraction of genomic regions switch between A and B compartments in different ways during organismal development and cellular differentiation \cite{Eijy Nagai et al. 2019 (Analyzing the 3D chromatin organization coordinating with gene expression regulation in B-cell lymphoma), Dixon et al. 2015 (Chromatin architecture reorganization during stem cell differentiation), Zhang et al. 2019 (Transcriptionally active HERV-H retrotransposons demarcate topologically associating domains in human pluripotent stem cells), Luo et al. 2020 (Reorganized 3D genome structures support transcriptional regulation in mouse spermatogenesis)}. Such compartment switching can move individual loci between permissive and repressed states along differentiation pathways, in part explaining regulatory differences between different cell types \cite{Lin et al. 2012 (Global changes in nuclear positioning of genes and intra- and inter-domain genomic interactions that orchestrate B cell fate), Dileep et al. 2019 (Rapid irreversible transcriptional reprogramming in human stem cells accompanied by discordance between replication timing and chromatin compartment)}. Some similar results have been observed in TADs, suggesting TAD locations and intra- and inter-TAD contact frequencies change during cellular development \cite{Chathoth and Zabet 2019 (Chromatin architecture reorganization during neuronal cell differentiation in Drosophila genome), Dixon et al. 2015 (Chromatin architecture reorganization during stem cell differentiation), Fraser et al. 2015 (Hierarchical folding and reorganization of chromosomes are linked to transcriptional changes in cellular differentiation), Bonev et al. 2017 (Multiscale 3D genome rewiring during mouse neural development), Chen et al. 2019 (Spatial genome re-organization between fetal and adult hematopoietic stem cells)}, although the extent of these alterations is less clear given variance in TAD identification \cite{Dali and Blanchette 2017 (A critical assessment of TAD prediction tools)}. Despite uncertainty about the role of TAD variance in cellular differentiation, it is clear that TADs play an important role in genome organization and function. Genes located within the same TAD can have strongly correlated expression patterns and are often co-regulated during cell differentiation \cite{Nora et al. 2012 (Spatial partitioning...), Zhan et al. 2017 (Reciprocal insulation analsis of Hi-C data shows that TADs represent a functionally...), Ramirez et al. 2018 (High-resolution TADs reveal DNA sequences underlying genome organization in flies)}. TAD boundaries are strongly correlated with replication-timing domain boundaries \cite{Pope et al. 2014 (Topologically associating domains are stable units of replication-timing regulation)}, and are enriched for insulator elements such as CCCTC-binding factor (CTCF) \cite{Dixon et al. 2012, Rao et al. 2014}. Disruptions to normative TAD structure have also been implicated in a number of human pathologies \cite{Lupianez et al. 2016 (How alterations of chromatin domains result in disease), Ibn-Salem et al. 2014 (Deletions of chromosomal regulatory boundaries are associated with congenital disease), Franke et al. 2016 (Formation of new chromatin domains determines pathogenicity of genomic duplications)}. Similarly, 3D genome organization at the lowest scale (i.e. individual gene-CRE loops) affects gene expression and regulation \cite{Greenwald et al 2019 (Subtle changes in chromatin loop contact propensity are associated with differential gene regulation and expression), }, and characterizing these interactions helps connect genetic variation with human trait and disease variation \cite{Gamazon et al. (Using an atlas of gene regulation across 44 human tissues to inform complex disease- and trait-associated variation), Montefiori et al. 2018 (A promoter interaction map for cardiovascular disease genetics), Zhao et al. 2020 (Molecular mechanisms of coronary disease revealed using quantitative trait loci for TCF21 binding, chromatin accessibility, and chromosomal looping), Mumbach et al 2017 (Enhancer connectome in primary human cells identifies target genes of disease-associated DNA elements)}. While GWAS have done well to identify variants associated with many human diseases, analysis of chromatin conformation capture data is necessary to understand understand how these variants exert their effects, which genes and loci they interact with, and, consequently, what therapeutic options may be effective \cite{Fadason et al. 2018 (Chromatin interactions and expression quantitative rait loci reveal genetic drivers of multimorbidities)}. In one particularly compelling example, researchers long thought that an obesity-associated mutation in an intron of the \textit{FTO} gene increased risk of obesity and type 2 diabetes by affecting the gene itself \cite{Grunnet et al. 2009 (Regulation and function of FTO mRNA expression in human skeletal muscle and...)}. This was not an unreasonable assumption, especially since follow-up studies found \textit{FTO} expression levels affect body mass in mice \cite{Church et al. 2010 (Overexpression of FTO leads to increased food intake...), Fischer et al. 2009 (Inactivation of the FTO gene protects from obesity), Gao et al. 2010 (The fat mass and obesity associated gene FTO functions in the brain...)}. Only through the application of chromosome conformation capture was it finally discovered that the variant in question actually regulates the expression of a transcription factor gene several megabases away, \textit{IRX3} \cite{Smemo et al. 2014 (Obesity-associated variants within FTO form long-range functional connections with IRX3)}. There are countless other examples where integration of Hi-C data has aided in identification of novel pathways and genes involved in the progression of various human pathologies \cite{Martin et al. 2015 (Capture Hi-C reveals novel candidate genes and complex long-range interactions with related autoimmune risk loci), Martin et al. 2016 (Identifying Causal Genes at the Multiple Sclerosis Associated Region 6q23 Using Capture Hi-C), Matoba et al. 2020 (Mapping Alzheimer's Disease Variants to Their Target Genes Using Computational Analysis of Chromatin Configuration), Wu and Pan 2018 (Integration of Enhancer-Promoter Interactions with GWAS Summary Results Identifies Novel Schizophrenia-Associated Genes and Pathways)}. Without a doubt, collecting and analyzing chromosome conformation capture data across species and cell types represents an exciting novel frontier in broadening our understanding of gene regulation, development, and evolution.

Overall, there is a dearth of comparative studies examining 3D genome architecture across species. In Chapter 2, I address this gap by collecting, integrating, and analyzing Hi-C and RNA-seq data from human and chimpanzee iPSCs. In Chapter 3, I challenge the prevailing notion that TADs are highly conserved across species by critically analyzing the existing data that support this claim. In the final chapter, I discuss the evolutionary and gene regulatory implications of my findings, before providing some perspective on the state of the 3D genome field and recommendations for future research directions.