\chapter{Reorganization of 3D genome structure may contribute to gene regulatory evolution in primates}\label{ch:ch02}

\section[Abstract]{Abstract\footnotemark}

\footnotetext{Citation for chapter: Eres IE, Luo K, Hsiao CJ, Blake LE, Gilad Y. 2019. Reorganization of 3D genome structure may contribute to gene regulatory evolution in primates. PLoS Genetics 15, e1008278. doi:10.1371/journal.pgen.1008278}

A growing body of evidence supports the notion that variation in gene regulation plays a crucial role in both speciation and adaptation. However, a comprehensive functional understanding of the mechanisms underlying regulatory evolution remains elusive. In primates, one of the crucial missing pieces of information towards a better understanding of regulatory evolution is a comparative annotation of interactions between distal regulatory elements and promoters. Chromatin conformation capture technologies have enabled genome-wide quantifications of such distal 3D interactions. However, relatively little comparative research in primates has been done using such technologies. To address this gap, we used Hi-C to characterize 3D chromatin interactions in induced pluripotent stem cells (iPSCs) from humans and chimpanzees. We also used RNA-seq to collect gene expression data from the same lines. We generally observed that lower-order, pairwise 3D genomic interactions are conserved in humans and chimpanzees, but higher order genomic structures, such as topologically associating domains (TADs), are not as conserved. Inter-species differences in 3D genomic interactions are often associated with gene expression differences between the species. To provide additional functional context to our observations, we considered previously published chromatin data from human stem cells. We found that inter-species differences in 3D genomic interactions, which are also associated with gene expression differences between the species, are enriched for both active and repressive marks. Overall, our data demonstrate that, as expected, an understanding of 3D genome reorganization is key to explaining regulatory evolution.

\section{Introduction}\label{ch02-introduction}
A growing body of evidence indicates that variation in gene regulation plays a key role in phenotypic divergence between species \cite{Britten.1971, King.1975, Carroll.2005, Gilad.2006, Wray.2007, Blekhman.2008, Karaman.2003}. Inferring the causal relationship between inter-species regulatory differences and phenotypic differences between species remains challenging, but compelling examples of regulatory adaptations have been published in a large number of species, including primates \cite{Prabhakar.2008, Babbitt.2010, Warner.2009, Loisel.2006, Rockman.2005, Pollard.2006}. The molecular mechanisms that underlie regulatory adaptation have also been the focus of much research. Studies in mice, flies, yeast, and primates have revealed that expression divergence between species is often driven by mutations or epigenetic modifications within cis-regulatory elements (CREs), rather than trans elements (e.g. transcription factors \cite{Wray.2007, Prabhakar.2008, Babbitt.2010, Warner.2009, Loisel.2006, Rockman.2005, Pennacchio.2013, Ong.2011}). This makes intuitive sense, because transcription factors can operate broadly across multiple functional contexts and throughout the genome (affecting many genes), whereas CREs often have more specific functional outcomes \cite{Carroll.2005, Wilson.2009}.

The ability to measure epigenetic marks, chromatin structure, and other functional genomic data has enabled us to identify and classify CREs into different types of regulators with distinct effects on gene expression (e.g. enhancers, silencers, insulators) \cite{Allis.2016, consortium.2012a, Won.2008}. Despite significant advances in our ability to identify and predict the functional role of CREs, we still lack a comprehensive characterization of the functional relationships between CREs and the genes they regulate. In many cases, we still do not know which genes are regulated by which CREs, or when and how often these relationships change. Connecting CREs to their target genes is crucial for understanding how regulatory architecture changes in response to different spatial, temporal and organismal contexts \cite{Ong.2011, Allis.2016, consortium.2012a, Won.2008, Levine.2010, Dowell.2011, Fagerberg.2014}. Ultimately, the effects of CREs on gene expression are likely to depend on which promoter(s) they contact, which is inherently related to the 3D structure of the genome \cite{Fukaya.2016, Huang.2017}.

The proximity and frequency of CRE-gene contacts can be measured in vivo using chromosome conformation capture techniques \cite{Dekker.2002}. Chromosome conformation affects how genes are expressed within a cell \cite{Babaei.2015, Gerstein.2012, Lan.2012, Botta.2010, Dong.2010, Homouz.2013, Jin.2013, Rao.2017}. For example, 3D genome structures may bring linearly distant loci into close proximity, connecting genes with CREs \cite{Shen.2012, Rieder.2012, Cremer.2001, Sexton.2007, Dekker.2008, Misteli.2007, Kosak.2004}. Expressed genes have been observed to spatially localize with distant CREs in 3D FISH experiments \cite{Rieder.2012, Schoenfelder.2010}. The latest chromosome conformation capture based technique, Hi-C, pairs the original method's proximity-based ligation with high-throughput sequencing to identify DNA-DNA contacts on a genome-wide scale \cite{Lieberman-Aiden.2009}. With enough sequencing coverage, Hi-C data can ultimately yield a comprehensive map of the 3D structure of an entire genome at high resolution \cite{Varoquaux.2014}.

Divergence in 3D genome structure may lead to regulatory evolution and ultimately to adaptation of new phenotypes. Currently, however, there are only a small number of comparative Hi-C data sets that can be used to test this notion \cite{Rudan.2015, Rao.2014, Dixon.2012}, and even fewer comparative data sets in primates \cite{Darrow.2016, Lazar.2018}. Most Hi-C studies to date have focused primarily on variation in chromatin contact frequencies within a single species \cite{Lieberman-Aiden.2009, Schmitt.2016, Kim.2017, Li.2015}. The few comparative Hi-C studies published to date typically draw comparisons between distantly related species (such as human and mouse \cite{Rao.2014, Dixon.2012}), use cancerous or otherwise transformed cell lines \cite{Rao.2014}, and rely on low resolution genome-wide Hi-C data (typically collecting 100-600M reads from most samples \cite{Rudan.2015, Dixon.2012, Dong.2017}). These comparative studies typically collect data in only a single individual from each species, and often compare contacts that are inferred from Hi-C libraries with large differences in read depth between species, a property that leads to differences in power to infer 3D genome structures at multiple scales \cite{Rudan.2015, Rao.2014, Dixon.2012}.

Thus, to conduct a comparative Hi-C study in primates and address these challenges, we collected high resolution Hi-C data from iPSCs derived from four human and four chimpanzee individuals. The human and chimpanzee genomes share a high degree of synteny \cite{Yunis.1982, Yunis.1980, Scally.2012, Kehrer-sawatzki.2007, Catacchio.2018, Lee.2016}, thus allowing us to consider a comparison of both low and high order chromatin interactions. Using our data, we were able to characterize `lower-order' locus-locus contacts and to infer `higher-order' structural features, such as TADs and TAD boundaries. We also quantified gene expression levels using RNA-seq data from the same eight cell lines. We considered our data with existing functional annotations, including histone marks and chromatin accessibility data, and evaluated the extent to which inter-species variation in 3D genome structure and epigenetic profiles are associated with gene expression divergence between humans and chimpanzees.

\section{Results}\label{ch02-results}

We performed in situ Hi-C as previously described \cite{Rao.2014} on a sex-balanced panel of four human and four chimpanzee integration-free iPSC lines that were previously generated and quality-checked by the Gilad lab \cite{Romero.2015}. Using HiCUP \cite{Wingett.2015} and HOMER \cite{Heinz.2010} (see Methods), we obtained genome-wide Hi-C contact maps at 10 kb resolution for all eight individuals, with each map containing approximately one billion sequencing reads. Since there is currently no gold standard for Hi-C normalization and statistical modeling, we also used an alternative method, Juicer \cite{Durand.2016}, to confirm that our results are robust with respect to the choices of normalization schemes and modeling (\ref{fig:figS1}{\textendash}\ref{fig:figS6} Figs). We also demonstrated the robustness of our results by performing certain analyses using different resolutions (from 10 kb to 500 kb). In the main text we report the results obtained using the HOMER pipeline at 10 kb resolution. Results using the alternative pipelines are shown in the supplement.

We used HOMER to independently classify between 779,503{\textendash}883,438 contacts (P \textless 0.01) in the Hi-C data obtained from each individual (genomic coordinates of all contacts in all individuals are provided in S1{\textendash}S8 Tables). We define a `contact' as a pair of 10 kb regions which we observed to be in physical proximity more often than expected by chance. Throughout the paper, we refer to Hi-C contacts as `lower-order' or `pairwise' interactions in order to distinguish them from higher-order, chromosome-scale structures (i.e. TADs and TAD boundaries).

Our goal was to compare Hi-C contacts between humans and chimpanzees. One intuitive approach to do so might be to identify the orthologous locations of each contact in the two species and classify such contacts as shared or unshared. However, this could lead to an inflated estimate of inter-species differences due to incomplete power to identify contacts in one species or the other. Instead, we collected the coordinates of all contacts identified in at least one individual into a single database. For each contact in the database, we independently identified the pair of corresponding orthologous regions in the human and chimpanzee genomes (using reciprocal searches, to avoid bias). Using this approach, we excluded about 18\% of contacts because we failed to identify clear orthologous regions in the genomes of the two species (see Methods and S9 Table). Following the orthology based filtering, we extracted the normalized contact frequencies (log2 observed:expected read count ratios) for all pairs of loci in the database, regardless of whether they were classified as contacts by HOMER. Thus, our analysis is not biased by potential differences in power to detect contacts in one species over the other. That said, we observed that the variance in contact frequency was lower for interactions that were independently identified in a greater number of samples, regardless of species (\ref{fig:figS7} Fig). We thus filtered out interactions that were independently classified as significant in fewer than four individuals. This approach allowed us to compare contact frequencies between species for 347,206 interactions while largely sidestepping the problem of incomplete power.

\subsection{Inter-species differences in 3D genomic interactions}
We used limma \cite{Smyth.2004} to perform pairwise cyclic loess normalization and minimize the effects of technical variables on our data (\ref{fig:figS8} Fig). Following normalization, principal components analysis (PCA) and unsupervised hierarchical clustering of the Hi-C data revealed that, as expected, samples cluster by species (Fig \ref{fig:fig1}).

\begin{figure}
\centering
\includegraphics[width=4in]{img/fig1.PNG}
\caption[General patterns in Hi-C data.]{\textbf{General patterns in Hi-C data.} (A) Principal components analysis (PCA) of HOMER-normalized interaction frequencies for the union of all contacts in humans (triangles) and chimpanzees (circles). PC1 is highly correlated with species (r = 0.98; P {\textless} 10\textsuperscript{-5}). (B) Unsupervised hierarchical clustering of the pairwise correlations (Pearson's r\textsuperscript{2}) of HOMER-normalized interaction frequencies at 10 kb resolution. The first letter in the labels demarcates the species (H for human and C for chimpanzee), and the following symbols indicate sex (male, M or female, F) and batch (1 or 2).}
\label{fig:fig1}
\end{figure}

To identify inter-species differences in contact frequencies, we analyzed the data using a linear model with fixed effects for species, sex, and processing batch (see Methods). At an FDR of 5\%, we classified 13,572 contacts (about 4\%) as having differential normalized contact frequency between humans and chimpanzees. Analysis of the orthologous regions anchoring these contacts suggested that approximately 4,000 of these differences might be explained by large inter-species differences in distance between mates of a contact pair (because read count is correlated with distance between the mates; see Methods and \ref{fig:figS9} Fig). We thus conservatively excluded locus pairs whose distance varied by more than 20 kb across species. Ultimately, we classified with confidence 9,661 Hi-C contacts (of 292,070; about 3.3\%) with a significant difference in normalized contact frequency between the two species. We refer to these contacts as inter-species differentially contacting (DC) regions (S10 Table). Our observations thus suggest that lower-order contacts are generally conserved between humans and chimpanzees. That said, if we assume that all of the contacts we filtered out (either due to lack of orthology or because the distance between the anchor regions differed across species) are in fact DC, divergence in contact frequency would have been observed for 16\% of the Hi-C contacts (assuming similar properties to the current data set). However, we find it more likely that a large subset of the contacts we excluded are not truly DC, but, rather, not comparable between the species due to differences in genome assembly quality.

Across all DC regions, 55\% exhibited a higher contact frequency in chimpanzees, while 45\% showed a higher frequency in humans (Fig \ref{fig:fig2}A, see Fig \ref{fig:fig3} and \ref{fig:figS10} Fig for examples). We observed that some chromosomes were associated with greater asymmetry in inter-species contact frequencies than others (Fig \ref{fig:fig2}B). Greater asymmetry seems to be present more often in chromosomes with large inter-species rearrangements. Specifically, in our data, 8 of the 9 chromosomes with known large-scale pericentric inversions between the species (1, 4, 5, 9, 12, 15, 16, 17, and 18; \cite{Yunis.1982, Yunis.1980, Kehrer-Sawatzki.2005, Nickerson.1998, Waterson.2005, Dennehey.2004}) show particularly strong asymmetry in inter-species contact frequencies. We also observed asymmetry in inter-species contact frequencies in human chromosome 2, a fusion of the ancestral chromosomes giving rise to chimpanzee chromosomes 2A and 2B \cite{Waterson.2005}, as well as in chromosome 7, which has the highest number of un-localized sequences of any chromosome in the panTro5 genome.

\begin{figure}
\centering
\includegraphics[width=6in]{img/fig2.PNG}
\caption[Linear modeling reveals large-scale chromosomal differences in contact frequency.]{\textbf{Linear modeling reveals large-scale chromosomal differences in contact frequency.} (A) Volcano plot of log\textsubscript{2} fold change in contact frequency between humans and chimpanzees (x-axis) against Benjamini-Hochberg FDR (y-axis), after filtering non-orthologus regions (results for unfiltered data are plotted in S9 Fig). Data are colored by the species in which the contact was originally identified as significant. (B) Per-chromosome volcano plot using the same legend as in A. P-values provided for a binomial test of the null that inter-species differences in contact frequencies are evenly distributed. The percentage of contacts with significant higher frequency in each species is noted.}
\label{fig:fig2}
\end{figure}

\begin{figure}
\centering
\includegraphics[width=6in]{img/fig3.PNG}
\caption[Examples of DC and non-DC interactions.]{\textbf{Examples of DC and non-DC interactions.} (A) PyGenomeTracks plots \cite{Ramirez.2018} of a chromosome 8 interaction between bins 130kb away for human (left panel) and chimpanzee (right). The bin pair tested is indicated by a black star, and was found to be DC between species. (B) Same as A, but for a conserved (non-DC) interaction on chromosome 17 separated by 100kb.}
\label{fig:fig3}
\end{figure}

Next, we turned our attention to higher-order chromosomal structures by characterizing TADs in each species. Previous studies indicate that the human and chimpanzee genomes share a high degree of synteny \cite{Yunis.1982, Yunis.1980, Scally.2012, Kehrer-sawatzki.2007, Catacchio.2018, Lee.2016}, a property we confirmed by tiling each genome into various bin sizes and using a reciprocal best hits liftOver method to identify syntenic regions (see Methods and \ref{fig:figS11} Fig). To infer steady-state TAD structures, we pooled reads across all individuals within each species to create `high-density consensus' Hi-C maps for humans and chimpanzees \cite{Durand.2016}. We used the Arrowhead algorithm at 10 kb resolution \cite{Durand.2016} to independently infer 11,298 TADs in humans and 10,505 TADs in chimpanzees (see Methods). We then used liftOver to identify orthologous genomic regions that corresponded to these TADs, and removed 10\% of domains for which orthology could not be identified (S11 and S12 Tables list the TADs identified in each species; S13 Table lists the orthologous locations of the combined TADs). Once orthology has been established, for each TAD, we considered the domain conserved in humans and chimpanzees when 90\% of the TAD interval overlapped reciprocally between species (see Methods). Using this approach, we found that only $\sim$43\% of TADs discovered in humans and chimpanzees are shared (Fig \ref{fig:fig4}A).

\begin{figure}
\centering
\includegraphics[width=6in]{img/fig4.PNG}
\caption[Higher-order chromosomal structure in humans and chimpanzees.]{\textbf{Higher-order chromosomal structure in humans and chimpanzees.} (A) Across different resolutions (x-axis), we plotted the number of shared and species-specific domains (y-axis) identified with Arrowhead \cite{Durand.2016} using the consensus map from each species (alternative approaches plotted in \ref{fig:figS12}{\textendash}\ref{fig:figS14} Figs). (B) Same as A, but for TAD boundaries instead of the domains themselves. Boundaries were defined as 15kb flanking regions at the edges of inferred Arrowhead domains. (C) Unsupervised hierarchical clustering of pairwise comparison of TADs across all individuals. These proportions were obtained using Arrowhead TAD inferences on each individual at 10kb resolution. Proportions indicated by color scale on right. Similar plots using analysis at different resolution are available in \ref{fig:figS12} and \ref{fig:figS14} Figs. (D) Similar to C, but for TAD boundaries instead of the domains themselves.}
\label{fig:fig4}
\end{figure}

The observation that TADs are generally not as conserved as practically all other regulatory phenotypes studied in humans and chimpanzees was unexpected. We thus thoroughly tested the robustness of this inference. To do so, we performed a large number of alternative analyses. We analyzed the data at different resolutions (from 10 kb to 500 kb---each time repeating the reciprocal liftOver analysis). We analyzed the data by considering, instead of pooled data, TADs identified independently in a single and in up to 4 individuals within each species (Fig \ref{fig:fig4}C and \ref{fig:figS12} Fig), and we did this across the different resolutions. We analyzed the data by classifying conservation based on the approach of Rao et al. \cite{Rao.2014} instead of relying on an overlap of 90\% of the domain; we analyzed the pooled data using panTro6 as a reference genome rather than the panTro5 assembly (\ref{fig:figS13} Fig). We analyzed the data by focusing on boundaries instead of the entire domains (Fig \ref{fig:fig4}B and \ref{fig:fig4}D); we used multiple alternative definitions of boundaries, and repeated this analysis across all resolutions and with boundaries identified in different numbers of individuals within species (\ref{fig:figS12} Fig). Finally, we identified TADs using an alternative algorithm, TopDom \cite{Shin.2016}, and repeated all of the alternative analyses mentioned above using this algorithm (\ref{fig:figS14} Fig).

The results of many of these alternative analyses are reported in the supplement (\ref{fig:figS10} and \ref{fig:figS12}{\textendash}\ref{fig:figS14} Figs). All of the alternative analyses produced consistent results and an inference that TADs and TAD boundaries are much less conserved between humans and chimpanzees than any other regulatory phenotype studied to date \cite{Pai.2011, Shulha.2012, Calarco.2007, Trizzino.2017, Prescott.2015, Kim.2011}. The Arrowhead analysis of TADs that are independently identified in four individuals within either species, at 10 kb resolution, where conservation is classified based on the less stringent approach of Rao et al. \cite{Rao.2014}, resulted in the highest estimate of conservation, with 78\% of domains and 83\% of boundaries shared between the species (\ref{fig:figS12}B and \ref{fig:figS12}D Fig). The restriction to TADs or boundaries identified in all 4 individuals of either species results in far fewer features that can be examined (\ref{fig:figS12} and \ref{fig:figS14} Figs), yet even in this analysis conservation of domains and boundaries is modest (see Fig \ref{fig:fig5} and \ref{fig:figS10} Fig for examples).

\begin{figure}
\centering
\includegraphics[width=6in]{img/fig5.PNG}
\caption[Examples of conserved and divergent TADs.]{\textbf{Examples of conserved and divergent TADs.}(A) A region on chromosome 1 with examples of both conserved and divergent Arrowhead \cite{Durand.2016} TAD inferences (black lines). Both the larger TADs seen in the chimpanzee map (right) appear to be conserved in the human map (left), whereas several of the TADs inferred in the human map are noticeably absent from the chimpanzee map. (B) A region on chromosome 11, once again showing examples of conserved and divergent Arrowhead TAD inferences (black lines). All the TADs seen in the human map (left) appear conserved in the chimpanzee map (right), whereas three smaller TADs inferred in the chimpanzee map are not found in the human map, suggesting divergence.}
\label{fig:fig5}
\end{figure}

\subsection{The relationship between inter-species differences in contacts and gene expression}

We previously collected RNA sequencing data from the same human and chimpanzee iPSC lines \cite{Pavlovic.2018}. We jointly analyzed the Hi-C and RNA-sequencing data to learn how often inter-species differences in 3D genomic contact frequencies are associated with inter-species differences in gene expression. We first identified 7,764 orthologous genes for which we have expression and Hi-C data anchored at a region that overlaps the gene's transcription start site (TSS; see Methods). A single genomic region that overlaps a TSS can have multiple contacts to other genomic regions. For the purpose of our analysis, we conservatively considered only the contact that shows the highest inter-species divergence for each gene.

We did not observe a correlation between gene expression and contact frequency when we considered data from all 7,764 genes. However, when we focused on the 1,401 genes classified as differentially expressed (DE) between humans and chimpanzees (at FDR ${\leq}$ 0.05), we observed an excess of both positive and negative correlations between inter-species differences in gene expression and inter-species differences in Hi-C contacts (\ref{fig:figS15} Fig). Indeed, genes whose TSS is associated with inter-species DC are more likely to be DE between species ($\chi^2$ test; P = 0.01; Fig \ref{fig:fig6}A and \ref{fig:fig6}B). The association between Hi-C contacts and gene expression divergence was somewhat stronger if instead of focusing on the contact with the highest divergence, we obtained a summary P-value \cite{Whitlock.2005} for testing the null hypothesis that there are no differences between the species in any of the contacts associated with the TSS for a given gene (P = 0.001; \ref{fig:figS20}C and \ref{fig:figS20}D Fig).

\begin{figure}
\centering
\includegraphics[width=6in]{img/fig6.PNG}
\caption[Differentially contacting Hi-C loci show enrichment for differentially expressed genes.]{\textbf{Differentially contacting Hi-C loci show enrichment for differentially expressed genes.} A) Enrichment of inter-species differentially expressed (DE) genes with corresponding differences in Hi-C contact frequencies (DC) between the species. The proportion of DC genes that are significantly DE (y-axis) is shown across a range of DC FDRs (x-axis). Colors indicate different DE FDR thresholds, and dashed lines indicate the proportion of DE genes expected by chance alone. (B) P values of Chi-squared tests of the null that there is no difference in proportion of DE genes among DC genes (y-axis), shown for a range of DC FDRs (x-axis). In both panels, DC regions were chosen to have the minimum FDR supporting inter-species difference in contact frequency. We plotted results using the weighted p-value combination instead of the minimum FDR in \ref{fig:figS20} Fig.}
\label{fig:fig6}
\end{figure}

A combined analysis of functional genomic data does not allow us to infer a direct causal relationship between chromatin contacts and gene expression patterns. Nevertheless, independent evidence strongly suggests that changes in 3D genomic structure can affect interactions between regulatory elements and promoters \cite{Rao.2014, Dily.2014, Chen.2017, Kagey.2010, Lupianez.2015}, which may ultimately drive differences in gene expression levels \cite{Rao.2017, Dily.2014, Chen.2017, Kagey.2010, Lupianez.2015, Siersbaek.2017, Niskanen.2018}. We thus sought to quantitatively estimate the extent to which inter-species DC might explain gene expression differences between the species in our data. To do so, we estimated and compared the effect of species on expression before and after accounting for the corresponding contact frequencies (see Methods; \cite{Baron.1986}).

Specifically, we performed a mediation analysis using linear models to assess the effect of contact on expression divergence (95\% confidence interval based on the Monte Carlo test of significance; see Methods). For approximately 8\% of DE genes (116/1401) we were able to reject the null hypothesis that the indirect effect is zero (\ref{fig:figS16} Fig). Taken together, these data suggest that a subset of inter-species differences in gene expression levels can be explained by divergence in Hi-C contacts.

\subsection{The chromatin and epigenetic context of inter-species differences in 3D genome structure}

Finally, we reasoned that species-specific contacts (i.e. significant DC regions) would be more likely to involve active, functional regulatory elements. This seems intuitive if one assumes most genomic contacts are functionally relevant, and not simply the result of pure noise. To test this hypothesis, we assessed the overlap between our Hi-C data and publicly available chromHMM annotations based on histone modification data from human embryonic stem cells \cite{consortium.2012a}. We assigned each Hi-C locus to an epigenetic state based on its maximum weighted base pair overlap with 15-state chromHMM annotations (see Methods and \ref{fig:figS17} Fig). Our approach to classify Hi-C regions with a functional assignment based on majority sequence overlap is arbitrary, but our conclusions are robust with respect to alternative approaches to analyze the Hi-C data (\ref{fig:figS4} and \ref{fig:figS17} Figs).

We found marked differences in the chromHMM annotations between genomic regions that are inferred to physically contact a promoter and those that do not contact a promoter (Fig \ref{fig:fig7}A and \ref{fig:fig7}B). For example, genomic regions in physical contact with a promoter are enriched with genic enhancer annotations ($\chi^2$ test; P = 0.0002, S14 Table), as might be expected. Perhaps more novel is the observation that inter-species DC regions were also enriched with genic enhancers, in contrast to regions that did not differ in contact frequency between the two species (P = 0.04, S14 Table). We note that this latter observation is not robust with respect to different annotations of enhancers, and we do not find this association if we simply combine all regions annotated as `enhancers' in the data set.

\begin{figure}
\centering
\includegraphics[width=4in]{img/fig7.PNG}
\caption[Overlap of epigenetic signatures and Hi-C contacts.]{\textbf{Overlap of epigenetic signatures and Hi-C contacts.} (A) Hi-C loci that do not make contact with promoters are ranked in order of decreasing DC FDR (x-axis). The y-axis shows cumulative proportion of chromHMM annotation assignments for all Hi-C loci at the given FDR or lower. (TssA-Active TSS, TSSBiv-Bivalent/Poised TSS, BivFlnk-Flanking Bivalent TSS/Enh, EnhBiv-Bivalent Enhancer, ReprPC-Repressed PolyComb, ReprPCWk-Weak Repressed PolyComb, Quies-Quiescent/Low, TssAFlnk-Flanking Active TSS, TxFlnk-Transcription at gene 5' and 3', Tx-Strong transcription, TxWk-Weak transcription, EnhG-Genic Enhancers, Enh-Enhancers, ZNF/Rpts-ZNF genes and repeats, Het-Heterochromatin). (B) Same as A, but only considering Hi-C loci making contact with promoter bins. Results of separating promoter-contacting bins between DE and non-DE genes can be seen in \ref{fig:figS19} Fig. (C) Density plot of the base pair overlap between different classes of Hi-C contact loci and H3K27ac. Histone mark data were obtained from ENCODE in experiments carried out in human iPSCs. We grouped contacts into 4 classes, indicated by color: those that show differential contact between species, those that show differential expression between species, those that show both, and those that show neither. We used pairwise t-tests to compare differences in the mean overlap among the four classes of Hi-C loci. (D) Same as C, but performed on H3K4me3 data obtained from ENCODE, collected in hESCs. Results with other histone marks can be seen in \ref{fig:figS18} Fig.}
\label{fig:fig7}
\end{figure}

We repeated the enrichment analysis of Hi-C regions using existing human iPSC histone mark data, including H3K27ac, H3K4me1, and H3K4me3, and h1-hESC histone mark data, including H3K27me3 and DNase I hypersensitivity sites (DHS \cite{consortium.2012a}). As expected, Hi-C regions in contact with a promoter showed greater overlap with DHS peaks than Hi-C regions that did not contact a promoter (t-test, P {\textless} 2.2 * 10\textsuperscript{-16}; \ref{fig:figS18}A Fig). When we focused on contacts involving a promoter, we found that inter-species DCs that are also associated with DE genes showed the largest overlap with DHS peaks, followed by DE genes that were not associated with DC regions (P = 0.01). Regions that were not associated with either DC or DE showed the least amount of overlap with DHS (P = 0.0006; \ref{fig:figS18}B Fig).

Remarkably, apart from the heterochromatic, repressive marker H3K27me3 (where the sign of the effect was the same, but the enrichment was not significant), Hi-C regions that are DC and are also associated with DE genes are more likely to overlap all other histone marks in our data set compared with Hi-C regions that are not DC and are not associated with a DE gene (all enrichment P {\textless} 0.03; Fig \ref{fig:fig7}C and \ref{fig:fig7}D, \ref{fig:figS18}C and \ref{fig:figS18}D Fig). In other words, inter-species DCs associated with DE genes are more likely to occur in genomic regions that are marked by histone modification, and are thus likely to have a regulatory function.

\pagebreak

\section{Discussion}\label{ch02-discussion}

In general, we observed lower-order, pairwise chromatin contacts in iPSCs to be conserved between humans and chimpanzees. We believe that this observation is intuitive, though we acknowledge that with only four individuals from each species, and given the challenges in identifying orthologous regions, we are likely to somewhat underestimate the degree of divergence in pairwise chromatin contacts.

In contrast to the conservation of lower order pairwise contacts, we did not find higher-order chromatin structures, such as TADs and TAD boundaries, to be generally conserved between human and chimpanzee iPSCs. Because this observation seems to contradict previous reports suggesting that TADs are strongly conserved across species \cite{Rao.2014, Dixon.2012}, we performed a large number of alternative analyses to demonstrate the robustness of our inference. Even in our most lenient analysis, we observed that only 78\% of domains are shared between humans and chimpanzees---a much lower conservation than observed for any other regulatory phenotypes between these two species (when similar sample sizes are considered; \cite{Pai.2011, Shulha.2012, Calarco.2007, Trizzino.2017, Prescott.2015}).

While all of the alternative analyses supported our inference, these analyses also demonstrated the known difficulty of robustly inferring TADs and TAD boundaries based on Hi-C data alone \cite{Ramirez.2018, Dali.2017}. Indeed, the algorithms used to infer TADs and TAD boundaries themselves are not very robust, as has been discussed previously \cite{Dali.2017, Forcato.2017}. Given our observations and the difficulty obtaining robust definitions of TADs and TAD boundaries, we carefully examined the previous evidence for high conservation of TADs between species.

The conclusion of our literature analysis is that the evidence for strong domain conservation is weak, and thus that our inference does not actually contradict previous data. A few of the studies typically cited as providing evidence for strong conservation of TADs across species did not actually perform a genome-wide assessment TADs, but inferred conservation based on a few examples \cite{Rudan.2015, Harmston.2017, Gomez-marin.2015}. Rudan et al. \cite{Rudan.2015}, for instance, reported functional conservation of TAD boundaries in liver cells from rhesus macaque, dog, rabbit, and mouse, but did not report the number or proportion of conserved regions they observed. Instead, they presented correlations of ~0.5 between contact frequencies across these species in subsets of contacts binned by the distance between mates, without further considering TADs or boundaries.

In contrast to studies that focused on specific examples, Dixon et al. \cite{Dixon.2012}, who originally described megabase-sized TADs at 40 kb resolution, reported that TAD boundaries were conserved in human and mouse embryonic stem cells. At a greater sequencing depth and finer resolution (1 kb), Rao et al. \cite{Rao.2014} observed TADs with a median size of 185 kb, and similar to Dixon et al., concluded that the domains were conserved in human and mouse B-lymphoblasts. However, the evidence for conservation in both studies is not strong. First, the actual conservation reported, though described as high, seems in fact to be modest: Dixon et al. reported that 54\% of human boundaries are shared with mouse (76\% if the comparison is reversed), and Rao et al. reported that 45\% of mouse domains are shared with human. Second, and more importantly, in both studies, conservation estimates were made unilaterally, by considering the proportion of TADs identified in the species for which they had less data that are also identified in the species for which they had more data. This approach results in an overestimate of sharing of domains because only the very strong TADs can be identified in the species with less data, and these are more likely to be shared across species. Indeed, if we perform a similar analysis using our own data (assessing sharing of the top 10\% of TADs identified in one species), we observe a much higher conservation (85\%). Conversely, if we use the data from Rao et al. to estimate reciprocal TAD sharing across species, conservation is even lower than originally reported, at $\sim$30\% instead of 45\%.

Thus, based on our analysis of the literature, we believe that the common notion that TADs are highly conserved in their placement across species is not well supported. Indeed, recent evidence from yeast \cite{Tjong.2012}, different Drosophila tissues \cite{Rowley.2017}, and plant species \cite{Dong.2017} suggests that TADs and TAD-like domains may not be particularly conserved, which raises questions about the stability of these higher-order structures and the significance of their role in the evolution of gene regulation across different lineages. However, the extent (or lack) of inter-species TAD conservation is difficult to falsify with existing data, partially because there is no standard method for identifying TADs, nor for comparing them across species \cite{Dali.2017, Forcato.2017}. The ability to reliably identify TADs also depends on the quality of the genome assemblies used, the approach for inferring synteny, sequencing depth and coverage, and various other parameters. We acknowledge that our estimates of inter-species differences in TADs may be somewhat inflated due to incomplete power to detect TAD structures in each genome. Unfortunately, the outputs of the available algorithms do not allow us to directly address this potential caveat in the same way we addressed incomplete power in the comparative analysis of lower-order interactions.

More generally, as many studies indicated \cite{Dali.2017, Yan.2017, Sauerwald.2018}, including ours, it is difficult to reconcile the visual examination of contact maps with TADs inferred based on algorithms. In our case (Fig \ref{fig:fig5} and \ref{fig:figS10} Fig), we found many examples where visual inspection naively suggests high conservation, but the algorithms do not indicate sharing of domains or boundaries. This is not surprising; a previous comprehensive analysis of numerous TAD algorithm inferences found very little concordance when compared to manual visual annotations of TADs \cite{Dali.2017}. Obviously, comparing all TAD inferences based on manual visual assessment is not feasible. Yet, the lack of stability of TAD algorithms means that it is possible that a better computational analysis will emerge and will indicate that domains or boundaries are indeed conserved. Currently, however, neither our own nor previously published data provides support for strong conservation of these structures.

\subsection{Contribution of variation in 3D genome structure to expression divergence}

We considered our Hi-C data along with gene expression data previously collected from the same cell lines \cite{Pavlovic.2018} and assessed the extent to which inter-species variation in 3D genome contacts could potentially explain gene expression divergence between species. Previous studies have observed that spatial co-expression of genes is associated with chromatin interaction profiles \cite{Babaei.2015, Dong.2010, Homouz.2013, Schoenfelder.2010, Duren.2017}. A number of studies have focused on differentially expressed genes following a treatment or perturbation and observed that such genes are often associated with corresponding differences in nearby chromatin contacts \cite{Dily.2014, Chen.2017}. Consistent with these reports, we found an enrichment of inter-species differences in pairwise chromatin contacts that involve promoters of differentially expressed genes between the species. Our observations are robust with respect to a range of data processing decisions and the statistical cutoffs we used. Under the common assumption that changes in chromatin contacts are more likely to explain differences in gene expression than vice versa, our results support the notion that species-specific 3D genomic contacts play an important role in the evolution of gene regulation.

Our observation that inter-species differences in pairwise genomic contacts are associated with regulatory evolution more than differences in large scale TAD boundaries is also consistent with previous reports. For example, Rao et al. \cite{Rao.2017} found that the degradation of cohesin, one of the proteins involved in maintaining TAD boundaries and large-scale loops, is associated with only modest effects on gene expression. In contrast, a number of other studies found strong correlations between differences in fine-scale genomic contacts and differences in the expression of nearby genes \cite{Rao.2014, Kagey.2010}.

Previous studies have identified a wide variety of regulatory phenotypes that contribute to inter-primate differences in gene expression levels \cite{Warner.2009, Loisel.2006, Pollard.2006, Pai.2011, Zhou.2014, Blekhman.2009, Cain.2011}; 3D genome conformation is only one of the putative upstream factors in the evolution of gene regulation. Our results argue for a model whereby inter-species differences in pairwise contact frequencies are among the main drivers of expression divergence between humans and chimpanzees. Given the low 10-kb resolution of our Hi-C data, it is likely that we have underestimated the contribution of inter-species variation in 3D genome structure to gene expression divergence between species. Future comparative Hi-C studies that sequence deeply enough to obtain higher, sub-kilobase resolutions, will allow researchers to resolve variation in contact frequency at even smaller scales, augmenting predictive power.

\subsection{Functional annotations}

Finally, we considered our data in the context of functional chromatin annotations available for the human genome. Previous studies have shown that 3D contact maps produced by Hi-C can be accurately recapitulated by epigenetic marks \cite{Pierro.2017, Zhu.2016}. Other reports have found enrichments for various chromatin accessibility and histone marks among interactions inferred from chromosome conformation capture data \cite{Roy.2015, Ron.2017}.

Our results corroborate and expand upon these findings. The differences we observed in chromHMM state assignments in our comparisons (namely, more active and less repressive states in promoter-involved contacts and contacts overlapping differentially expressed genes), provide additional support for the functional relevance of our inferences. We acknowledge that these differences could potentially be more pronounced with higher-resolution Hi-C data and with chromHMM inferences made from ChIP-seq experiments in the same cell lines. While our study design does not allow us to directly infer causality between chromatin interactions and gene expression, the functional enrichments we observed for different epigenetic marks suggest that 3D genome conformation may be one of the upstream elements in the chain of events driving the evolution of gene expression. Although this notion is intuitive to us and is consistent with our data, it is still possible that differences in epigenetic marks are the true drivers of divergence in gene expression levels and/or chromatin contacts between humans and chimpanzees.

Future studies integrating similar data types could explore these possibilities by examining epigenetic marks across species (only human data were available to us), which would enable researchers to polarize the regulatory differences in orthologous sequences between humans and chimpanzees. This would also allow for a sharper definition of the functional classes of inter-species differences in lower-order chromatin contacts.

\section{Materials and methods}

\subsection{Ethics statement}

We collected human fibroblasts with written informed consent obtained from all human participants under University of Chicago IRB protocol 11---0524. We obtained fibroblasts from chimpanzees from the Yerkes Primate Research Center of Emory University under protocol 006---12, in full compliance with IACUC protocols \cite{Romero.2015}. All experimental methods are in accordance with the Helsinki Declaration.

\subsection{Induced pluripotent stem cells (iPSCs)}

As described previously, the Gilad lab has derived panels of both human and chimpanzee iPSCs via episomal reprogramming \cite{Romero.2015}. To ensure their quality, we validated iPSCs from both species as pluripotent at high passages ({\textgreater}10). Quality control checks included an embryoid body assay confirming their ability to differentiate into all three germ layers, qPCR of endogenous transcription factors associated with pluripotency, PCR to confirm the absence of exogenous pluripotency genes (both from residual episomal plasmid or genomic integration), and PluriTest \cite{Muller.2011}, a bioinformatics classifier that assesses pluripotency based on gene expression data \cite{Romero.2015}. In the current study, we grew all cell lines in the same incubator in two passage-matched batches, which were also balanced across species and sex, in order to avoid batch effects in our data.

\subsection{In-situ Hi-C library preparation and sequencing}
We performed \textit{in situ} Hi-C with the restriction enzyme MboI, as previously described \cite{Rao.2014} on the iPSCs from both species. We grew cells in feeder-free conditions \cite{Nakagawa.2014} to approximately 80\% confluence before adding formaldehyde to crosslink the proteins mediating DNA-DNA contacts. We flash-froze pellets of 5 million cells each before beginning the in situ Hi-C protocol \cite{Rao.2014}. We used MboI to cut the DNA at each of its 4-bp recognition sites (GATC) throughout the genome. Ligation of proximal fragments with T4 DNA ligase yielded chimeric DNA molecules representing two distinct loci. Libraries were created in two balanced batches identical to the cell growth batches and sequenced (100bp paired-end) on an Illumina Hi-Seq 4000 at the University of Chicago Genomics Core Facility. To avoid batch effects resulting from differences in flow cells, libraries were sequenced across three lanes, each on separate flow cells balanced for species.

\subsection{Hi-C read mapping, filtering, and normalization}

We preprocessed, mapped, and filtered the resulting FastQ sequence files using HiCUP version 0.5.9 \cite{Wingett.2015}. We also used HiCUP to truncate the reads at ligation junctions. Thereafter, we used bowtie2 version 2.2.9 \cite{Langmead.2012} to independently map the two mates of paired-end sequences to either the hg38 or panTro5 genomes, and removed reads with low quality scores (MAPQ {\textless} 30). We carried out further HiCUP filtering as previously described based on an \textit{in silico} genome digest in order to remove experimental artifacts \cite{Wingett.2015}. We then used HOMER version 4.9.1, a foundational statistical analysis suite for Hi-C data \cite{Heinz.2010}, to tile the genome into a matrix of 10 kb bins and assign reads to their corresponding intersecting bins. We subsequently used HOMER to normalize Hi-C contact bins as previously described \cite{Heinz.2010}, accounting for known technical biases in Hi-C data. Finally, we called statistically significant interactions independently in each individual using HOMER, based on a null expectation of read counts falling into bins in a cumulative binomial distribution \cite{Heinz.2010}. We retained interactions with an unadjusted P value ${\leq}$ 0.01, the default recommendation by HOMER. As other studies have noted \cite{Durand.2016}, a traditional multiple testing correction paradigm is overly conservative for Hi-C data due to the high number of tests, and because the spatial nature of the data means that individual tests are highly correlated (and thus not independent).

\subsection{Creation of a union list of orthologous Hi-C contacts across species}

In order to ensure that the contact frequencies we compared across species were from representative orthologous sequences in humans and chimpanzees, we used liftOver with a reciprocal best hits method \cite{Ward.2014, Kent.2002} to transfer interaction bin coordinates across both genomes. For each called contact, we used liftOver to independently map the coordinates of the two anchor bins in the other species' genome, obtaining coordinates in both genomes for all contacts. We then rounded the coordinates to the nearest 10 kb bin, in order to align properly with a Hi-C bin. We required both anchor bins to have orthologous bins in the other species in order to retain a contact for comparison; statistics on the number of called contacts and the number retained after our liftOver procedure are available in S9 Table. In order to assess the extent of contacts lost due to lack of orthology, we also compared the retention of genome-wide 10 kb bins in both genomes with the retention of unique 10 kb bins found within each of our individuals. We found that our Hi-C bins tended to have a higher rate of orthologous mappability across species (S9 Table). For all contacts in this union list, we then extracted the HOMER-normalized interaction frequencies from each individual's 10 kb Hi-C matrix. Including interactions discovered in fewer than 4 individuals increased the variance in our data (\ref{fig:figS7} Fig). Therefore, we retained only the Hi-C contacts that were independently discovered by HOMER in at least 4 individuals, for a total of 347,206 interactions. As we describe in the next section, we also later filtered out contacts where the distance between bins showed a difference of {\textgreater} 20 kb across species, retaining 292,070 interactions.

\subsection{Linear modeling of Hi-C interaction frequencies}

In an effort to quantify inter-species differences in the Hi-C interaction frequency values, we used the following linear model:

\begin{equation} \label{eq:limma}
Y_{ij} = \beta_{0} + \beta_{sp}s_{i} + \beta_{sx}x_{j} + \beta_{btc}b_{i} + \epsilon_{ij}
\end{equation}

$Y_{ij}$ represents the observed Hi-C interaction frequency of a contact from individual j in species i. $\beta_{0}$ is the intercept. $\beta_{sp}$, $\beta_{sx}$, and $\beta_{btc}$ are effect sizes for species, sex, and batch, respectively, with their corresponding variables $s_{i}$, $x_{j}$, and $b_{i}$, and an error term $\epsilon_{ij}$. We used the R/Bioconductor package limma \cite{Smyth.2004, Law.2014} to test for inter-species differences in Hi-C interaction frequency. We applied Benjamini-Hochberg multiple testing correction and found 13,572 interaction pairs where the species term is significant at a 5\% false discovery rate (FDR).

Initial visualization of the linear modeling results for the species term revealed a stark asymmetry (\ref{fig:figS9}A Fig) suggesting that on a global level, the contacts identified as significant in chimpanzees were much stronger than those identified in humans. This was surprising to us; we reasoned that this asymmetry could be due to a technical factor. For example, liftOver conversion of genome coordinates between species to identify orthologous bins can create differences in both the Hi-C locus size and in the genomic distance between mates of a contact pair (mate-pair distance). We investigated the impact of these two factors on the proportion of contacts classified as differential across species in our data. We discovered that while changes in Hi-C locus size had little effect on the proportion of interspecies DCs, differences in mate-pair distances {\textgreater} 20 kb across species created a noticeable inflation in this proportion at an FDR of 5\% (\ref{fig:figS9}B Fig). We believe this makes intuitive sense, as bins that are farther apart will have fewer read counts due to the proximity-based ligation in Hi-C. Thus, a mate-pair distance difference across the genomes could induce what appears to be a differential contact, because the contact inherently has more read support in the species where the mates are closer. However, we note that it is impossible to ascertain the relative biological and/or technical relevance of the differences seen in these contacts. We thus took a conservative approach to minimize false positives and removed contacts with a {\textgreater}20 kb mate-pair distance difference between species from our downstream analyses (\ref{fig:figS9} Fig), accepting that the number of inter-species differences we observe may be underestimated.

\subsection{Identification of orthologous topologically associating domains (TADs) and boundaries}

We chose to perform TAD analyses on both individual-level data and on representative species consensus data. For our analysis comparing TAD boundaries on species consensus Hi-C maps, we combined all the preprocessed Juicer files from all our individuals within a species and used the \textit{juicer\_mega.sh} script \cite{Durand.2016} to create higher density contact maps for each species. We then ran the Arrowhead algorithm across resolutions to infer TADs, and then we extended the edges of TADs 7.5 kb in each direction to create 15 kb boundaries (accounting for imprecision in boundary inference). We used a reciprocal best hits liftOver strategy \cite{Ward.2014, Kent.2002} to obtain orthologously mappable TADs and boundaries. To confirm high synteny of large-scale linear genomic intervals between the species, we employed this same orthology analysis on genome-wide tilings of the hg38 and panTro5 genome assemblies, with varying window sizes created with bedtools \cite{Quinlan.2010} \textit{makewindows} (\ref{fig:figS11} Fig). In the case of TADs, we then assessed number of domains found in one species that were also found in the other species (conserved domains) with reciprocal bedtools \cite{Quinlan.2010} \textit{intersect {\textendash}c {\textendash}f 0.9 -r} calls. These parameters will only define a domain as overlapping if there is a domain in the other species such that each domain shares 90\% of their interval with the other. We used the larger of the two estimates of shared TADs across the species as the conserved domain count (to be conservative), and divided this by the sum of the conserved and species-specific domains identified in order to assess conservation. As an alternative analysis, we also employed the method previously described by Rao et al. \cite{Rao.2014}; namely, we called a TAD conserved in one species if it and a TAD from the other species displayed a Euclidean distance less than the smaller of 50 kb or half the given TAD's size. We analyzed boundary conservation using bedtools \textit{intersect {\textendash}c}, considering any overlap as indication of conservation (i.e. even a single base pair overlap of boundaries meant a boundary was classified as conserved).

To examine individual-level data and to ensure robustness of our results, we separately used both Arrowhead \cite{Durand.2016} and TopDom \cite{Shin.2016} (with \textit{window = 20}) across resolutions to call TADs independently in each individual sample. Though we performed essentially the same analyses on both outputs, it should be noted that Arrowhead provides nested TADs only, from which we inferred boundaries as described above, whereas TopDom provides separate domain and boundary inferences. We used a reciprocal best hits liftOver method \cite{Ward.2014, Kent.2002} to obtain a set of orthologous domains and boundaries. We assessed interspecies conservation by performing left outer joins (bedtools \textit{intersect {\textendash}loj}) of each individual's domains against all the others, once again requiring 90\% reciprocal overlap. We then took the average species-specific and shared domain counts across these individual comparisons to produce a single estimate of conservation (\ref{fig:figS12} and \ref{fig:figS14} Figs). The individuals' pairwise percentages of shared domains were used in hierarchical clustering analysis (Fig \ref{fig:fig4}C, \ref{fig:figS12} and \ref{fig:figS14} Figs). We also once again checked the robustness of our results using the conservation calling method from Rao et al. \cite{Rao.2014} described above. In the case of boundaries, we reasoned that, given the nested nature of the TADs, as well as variance between individuals in their exact placement, it would make sense to merge the boundaries (using bedtools \textit{merge}) in order to obtain a list of unique boundary elements. We added a column of individual identifiers to each set of boundaries and then merged all together, thereafter assessing conservation by examining what percentage of boundaries were independently found in both species out of the total set of unique boundaries. We also applied hierarchical clustering analysis to individual pairwise percentages of shared boundaries in this union merged file (Fig \ref{fig:fig4}D, \ref{fig:figS12} and \ref{fig:figS14} Figs). Further descriptions of these analyses can be found on our GitHub repository (/data/TADs folder), and 10 kb individual Arrowhead inferences are available in S17{\textendash}S24 Tables.

\subsection{Differential expression analysis}

Previously, the Gilad lab generated RNA-seq expression data on the same iPSC lines from this study (GEO accession GSE110471 \cite{Pavlovic.2018}). We computed reads per kilobase per million mapped reads (RPKM) for every gene, as the orthologous genes are not constrained to be the same length across species. We retained 11,074 genes that had at least half of the individuals (2 observations) in each species with log\textsubscript{2}RPKM ${\geq}$ 0.4. We then used the limma-voom pipeline with precision weights \cite{Smyth.2004, Law.2014} to test for differential expression across species, using a linear model including a species effect and a sex effect. Using this approach, we found 2,086 differentially expressed genes (at 5\% FDR).

\subsection{Broad integration of Hi-C and gene expression data}

We obtained the overlap between our gene expression data and our Hi-C data by applying bedtools \textit{overlap} \cite{Quinlan.2010} to the Hi-C loci and the first exon of each gene. Using a curated file of orthologous gene coordinates between humans and chimpanzees \cite{Pavlovic.2018}, we extracted a one-base-pair interval at the beginning of each first exon to use as a proxy for transcription start sites (TSSs).

As described in the main text, the difference in dimensionality between the two datasets presented a challenge. While every gene has only one expression value per individual, a given Hi-C locus can and frequently does make contact with many other loci. When a given gene overlapped a Hi-C locus making multiple contacts, we chose the contact with the smallest species term FDR (i.e. the most species-specific contact) in our DC analysis to represent the interaction frequency for that gene. Accordingly, we interpreted the FDR-adjusted P value for the chosen contact as the gene's differential contact significance. To examine correlations between normalized Hi-C contact frequency and log\textsubscript{2}RPKM gene expression, we considered the correlation between gene expression values across all 8 individuals with the corresponding interaction frequency values across the same 8 individuals.

\subsection{Enrichment of differential expression in differential contacts}

We examined the enrichment of differential expression in genes with differential contact (Fig \ref{fig:fig6}A and \ref{fig:figS20}C Fig) across a continuous range of DC FDRs and a discrete range of DE FDRs (1\%, 2.5\%, 5\%, 7.5\%, and 10\%). We used Pearson's chi-squared test to quantify significance of the enrichment at each FDR (Fig \ref{fig:fig6}B and \ref{fig:figS20}D Fig). We also examined the reciprocal enrichment; that is, DC enrichment amongst DE genes (\ref{fig:figS20}A and \ref{fig:figS20}B Fig).

\subsection{Assessing the quantitative contribution of Hi-C contact frequencies to gene expression levels}

We assessed the hypothesis that expression divergence may be mediated by contact frequency using linear models \cite{Baron.1986}. The intuition behind this approach is that the effect of species (X) on expression (Y) can be partitioned into its indirect effect on expression mediated through contact frequency (M) and its direct effect on expression. Therefore, a significant indirect effect would suggest that expression divergence is causally mediated by contact frequency. To test our mediation hypothesis, we computed the indirect effect of species on expression (X $\rightarrow$ M $\rightarrow$ Y: causal effect of X on Y through M) by taking the product of the effect of species on contact frequency ($\alpha$: X $\rightarrow$ M) and the effect of contact frequency on expression after controlling for species ($\beta$: M $\rightarrow$ Y). The indirect effect ($\alpha$*$\beta$) is conceptually equivalent to the difference between the effect of species on expression and the effect of species on expression after controlling for contact frequency, but is more mathematically tractable and commonly used in mediation analyses \cite{Sobel.1986, Sobel.1982, MacKinnon.2002}. We obtained $\alpha$ as the species effect size in a simple linear model attempting to predict Hi-C interaction frequency based solely on a species term. We estimated $\beta$ as the contact frequency effect size in a linear model predicting expression based on both species and contact frequency per gene. To determine statistical significance of the indirect effect, we applied the Monte Carlo test of significance to construct the 95\% confidence interval. The primary benefits of the Monte Carlo method are that it requires no distributional assumptions of the data and is robust against type I error in small samples \cite{MacKinnon.2004, Preacher.2012, Preacher.2004}. Thus, we choose the Monte Carlo test over Sobel test, the conventional approach to significance testing of mediation, which relies on the data following normal distribution \cite{Sobel.1986, Sobel.1982}.

\subsection{Integration with epigenetic annotations}

We obtained chromHMM 15-state model peak calls in human iPSC cells from ENCODE \cite{consortium.2012a} (S15 Table). We subsequently found the overlap between the human coordinates of our orthologous Hi-C contact loci and the chromHMM peak calls and quantified the extent of base pair overlap between each locus and all overlapping chromHMM peaks. We assigned each individual locus a single chromHMM annotation based on the peak with the highest base pair overlap with that locus. However, the distribution of overlaps of different chromHMM annotation peaks with our Hi-C bins were quite variable in size. To account for this, we normalized each annotation's overlap length in each locus by multiplying it by the reciprocal of its mean base pair overlap across all our bins (\ref{fig:figS17} Fig). After removing duplicate Hi-C loci, we then assigned individual loci to chromHMM annotations based on these normalized base pair overlaps. We started with a small set of the top ten most differentially contacting loci (i.e. the ten lowest FDR loci from our Hi-C linear modeling), and tabulated proportions of which annotations were represented amongst them. We then iteratively added the next-lowest FDR contact (i.e. two Hi-C loci at a time) to this tabulation, re-calculating proportions on the new set of contacts. We ran this same cumulative proportions analysis separately on contacts not overlapping promoters, contacts overlapping promoters, contacts overlapping promoters of DE genes, and contacts overlapping promoters of genes that were not DE (Fig \ref{fig:fig7}A and \ref{fig:fig7}B, \ref{fig:figS19} Fig).

We also obtained data on H3K4me1, H3K4me3, and H3K27ac collected in human iPS-18A cells, and data on H3K27me3 and DNase hypersensitivity sites collected in H1-hESCs, all from ENCODE \cite{consortium.2012a} (S15 Table). We used bedtools \textit{intersect} \cite{Quinlan.2010} to find the base pair overlap between each of these different marks and our Hi-C contact loci. We then removed duplicate Hi-C loci from the dataset and used a pairwise t-test to identify significant differences in the overlapping distributions for different sets of Hi-C classes (based on differential contact and differential expression, Fig \ref{fig:fig7}C and \ref{fig:fig7}D).

\section{Acknowledgments}

We thank the members of the Gilad, Nobrega, and Stephens labs for helpful discussions, particularly Matthew Stephens, Bryan J. Pavlovic, D\'ebora R. Sobreira, Abhishek Sarkar, and Lindsey E. Montefiori. We thank Natalia Gonzales for help editing the paper.

\subsection{Author contributions}

IEE and YG conceived of the study and designed the experiments. IEE performed the experiments. IEE analyzed the results (with input from CJH, KL, and LEB). YG supervised the project. IEE and YG wrote the paper. All authors read and approved the final manuscript.

\pagebreak
\clearpage

\section{Supplementary Figures}\label{ch02-supplementary-figures}

\begin{figure}[!htb]
\centering
\includegraphics[width=6in]{img/figS1.pdf}
\caption[Regulatory landscapes cluster by species, Juicer.]{\textbf{S1. Regulatory landscapes cluster by species, Juicer.} (A) Principal components analysis (PCA) of Juicer vanilla coverage (VC)-normalized interaction frequencies for the union of all contacts in humans (triangles) and chimpanzees (circles). PC1 is highly correlated with species (r = 0.89; P = 0.0004). (B) Unsupervised hierarchical clustering of the pairwise correlations (Pearson's r\textsuperscript{2}) of Juicer VC-normalized interaction frequencies at 10 kb resolution. The first letter in the labels demarcates the species (H for human and C for chimpanzee), and the following symbols indicate sex (male, M or female, F) and batch (1 or 2).}
\label{fig:figS1}
\end{figure}

\begin{figure}[!htb]
\centering
\includegraphics[width=6in]{img/figS2.pdf}
\caption[Linear modeling reveals large-scale chromosomal differences in contact frequency, Juicer.]{\textbf{S2. Linear modeling reveals large-scale chromosomal differences in contact frequency, Juicer.} (A) Volcano plot of log\textsubscript{2} fold change in contact frequency between humans and chimpanzees (x-axis) against Benjamini-Hochberg FDR (y-axis), after filtering non-orthologous regions. Data are colored by the species in which the contact was originally identified as significant. (B) Per-chromosome volcano plot using the same legend as in A. P-values provided for a binomial test of the null that inter-species differences in contact frequencies are evenly distributed. The percentage of contacts with significant higher frequency in each species is noted. Of note is that many of the same chromosomal asymmetries in contact strength observed here are in the same chromosomes as those observed in the HOMER-normalized data (Fig \ref{fig:fig2}).}
\label{fig:figS2}
\end{figure}

\begin{figure}[!htb]
\centering
\includegraphics[width=6in]{img/figS3.pdf}
\caption[Differentially expressed genes show enrichment for differential Hi-C contacts, Juicer.]{\textbf{S3. Differentially expressed genes show enrichment for differential Hi-C contacts, Juicer.} (A) Enrichment of inter-species differentially expressed (DE) genes with corresponding differences in Hi-C contact frequencies (DC) between the species. The proportion of DC genes that are significantly DE (y-axis) is shown across a range of DC FDRs (x-axis). Colors indicate different DE FDR thresholds, and dashed lines indicate the proportion of DE genes expected by chance alone. (B) P values of Chi-squared tests of the null that there is no difference in proportion of DE genes among DC genes (y-axis), shown for a range of DC FDRs (x-axis). In both panels, DC regions were chosen to have the minimum FDR supporting inter-species difference in contact frequency. (C) Same as A, but this time, a weighted p-value combination technique \cite{Whitlock.2005} was used to integrate each Hi-C bin's DC FDR across all of its contacts. (D) Same as B, but for the weighted p-value combination instead of the minimum FDR contact.}
\label{fig:figS3}
\end{figure}

\begin{figure}[!htb]
\centering
\includegraphics[width=6in]{img/figS4.pdf}
\caption[Dynamics of chromHMM state among significant Hi-C contacts, Juicer.]{\textbf{S4. Dynamics of chromHMM state among significant Hi-C contacts, Juicer.} (A) Hi-C loci that do not make contact with promoters are ranked in order of decreasing DC FDR (x-axis). The y-axis shows cumulative proportion of chromHMM annotation assignments for all Hi-C loci at the given FDR or lower. (TssA-Active TSS, TSSBiv-Bivalent/Poised TSS, BivFlnk-Flanking Bivalent TSS/Enh, EnhBiv-Bivalent Enhancer, ReprPC-Repressed PolyComb, ReprPCWk-Weak Repressed PolyComb, Quies-Quiescent/Low, TssAFlnk-Flanking Active TSS, TxFlnk-Transcription at gene 5' and 3', Tx-Strong transcription, TxWk-Weak transcription, EnhG-Genic Enhancers, Enh-Enhancers, ZNF/Rpts-ZNF genes and repeats, Het-Heterochromatin). (B) Same as A, but only considering Hi-C loci making contact with promoter bins. (C) Same as B, but only considering Hi-C loci making contact with promoters of genes that are not differentially expressed (DE). (D) Same as C, but only considering Hi-C loci making contact with promoters of genes that are differentially expressed (DE).}
\label{fig:figS4}
\end{figure}

\begin{figure}[!htb]
\centering
\includegraphics[width=6in]{img/figS5.pdf}
\caption[Overlap of activating and repressive histone marks among Hi-C contacts, Juicer.]{\textbf{S5. Overlap of activating and repressive histone marks among Hi-C contacts, Juicer.} (A) Density plot of the base pair overlap between different classes of Hi-C contact loci and H3K27ac. Histone mark data were obtained from ENCODE in experiments carried out in human iPSCs. We grouped contacts into 4 classes, indicated by color: those that show differential contact between species, those that show differential expression between species, those that show both, and those that show neither. We used pairwise t-tests to compare differences in the mean overlap among the four classes of Hi-C loci. Unlike in the HOMER-normalized data, we do not observe statistically significant differences in overlaps with H3K27ac between different locus classes. This may reflect the previous observation that the hiccups algorithm for assigning statistical significance of loops in Hi-C data is much more conservative than HOMER's significance calling method \cite{Forcato.2017}. (B) Same as A, but performed on H3K4me3 data obtained from ENCODE, collected in hESCs. (C) Same as A and B, but performed on H3K4me1 data obtained from ENCODE, collected in human iPSCs. (D) Same as A, B, and C, but performed on H3K27me3 data obtained from ENCODE, collected in human iPSCs.}
\label{fig:figS5}
\end{figure}

\begin{figure}[!htb]
\centering
\includegraphics[width=6in]{img/figS6.pdf}
\caption[Gene expression variance is explained by chromatin contacts for 5\% of DE genes, Juicer.]{\textbf{S6. Gene expression variance is explained by chromatin contacts for 5\% of DE genes, Juicer.} Plot of the species effect size in DE genes between models before (x-axis) and after (y-axis) conditioning on contact frequency. The Monte Carlo test of significance was used to construct the 95\% confidence interval and evaluate the significance of the indirect effect (species' effect on expression mediated through contact). Amongst DE genes, 5\% (15/299) of genes showed a statistically significant effect of Hi-C contacts on expression levels (i.e. their 95\% confidence interval does not include zero).}
\label{fig:figS6}
\end{figure}

\begin{figure}[!htb]
\centering
\includegraphics[width=6in]{img/figS7.pdf}
\caption[Variance in interaction frequency as a function of the number of individuals in which a significant interaction is independently discovered.]{\textbf{S7. Variance in interaction frequency as a function of the number of individuals in which a significant interaction is independently discovered.} (A) Boxplots of variance in contact frequency across all 8 individuals on the y-axis, binned by the number of individuals in which an interaction is independently called significant on the x-axis. (B) Same as A, but zoomed in on the y-axis to visualize finer-scale variation.}
\label{fig:figS7}
\end{figure}

\begin{figure}[!htb]
\centering
\includegraphics[width=6in]{img/figS8.pdf}
\caption[Distributions of HOMER-normalized interaction frequencies are remarkably similar across species.]{\textbf{S8. Distributions of HOMER-normalized interaction frequencies are remarkably similar across species.} (A) Histogram of log\textsubscript{2}(observed/expected) HOMER-normalized interaction frequencies in all four human samples used in this study, after applying pairwise cyclic loess normalization with limma \cite{Smyth.2004}. (B) Same as A, but in chimpanzees.}
\label{fig:figS8}
\end{figure}

\begin{figure}[!htb]
\centering
\includegraphics[width=6in]{img/figS9.pdf}
\caption[Volcano plot asymmetry quality control.]{\textbf{S9. Volcano plot asymmetry quality control.} (A) Volcano plot of log\textsubscript{2} fold change in contact frequency between humans and chimpanzees (x-axis) against Benjamini-Hochberg FDR (y-axis). This plot shows data only filtered for independent discovery in at least 4 individuals. Data are colored by the species in which the contact was originally identified as significant. (B) Scatter plot of sets of Hi-C contacts, with proportion of contacts significant in our linear modeling of interaction frequency shown based on color. Contacts are binned by mate-pair distance differences (y-axis) and bin size differences (x-axis). Circle size is proportional to the size of the set of Hi-C contacts falling into each criteria. Red indicates that the data were filtered out after this step, and blue/purple indicates that the data were retained for further analysis. (C) Volcano plot as in A, but after removing contacts with large mate-pair distance differences across the species.}
\label{fig:figS9}
\end{figure}

\begin{figure}[!htb]
\centering
\includegraphics[width=6in]{img/figS10.1.pdf}
\caption[Further visual examples of DC and non-DC interactions; conserved and divergent TADs.]{\textbf{S10. Further visual examples of DC and non-DC interactions; conserved and divergent TADs.} (A) PyGenomeTracks plots \cite{Ramirez.2018} of a chromosome 19 interaction between bins 80 kb away for human (left panel) and chimpanzee (right panel). The bin pair tested is indicated by a black star, and was found to be DC between species. (B) Same as A, but for a conserved (non-DC) interaction on chromosome 1 separated by 100kb...(continued on next page)}
\label{fig:figS10}
\end{figure}

\begin{figure}[!htb]
\includegraphics[width=6in]{img/figS10.2.pdf}
\contcaption{(continued from previous page) (C-H) Examples of contact maps (created with PyGenomeTracks \cite{Ramirez.2018}) and Arrowhead-inferred TAD structures (black lines) in humans (left) and chimpanzees (right), across a number of different chromosomes. In most examples, inference based on the algorithm indicates shared and species-specific domains, yet these are difficult to ascertain based on visual inspection, as discussed.}
\end{figure}

\begin{figure}[!htb]
\centering
\includegraphics[width=6in]{img/figS11.pdf}
\caption[Synteny of large scale linear genomic intervals between human and chimpanzee.]{\textbf{S11. Synteny of large scale linear genomic intervals between human and chimpanzee.} (A) Across different window sizes (x-axis) for a genome-wide tiling of hg38, we plotted the number of total and syntenic linear intervals (y-axis), identified using the reciprocal best hits liftOver method \cite{Ward.2014, Kent.2002} we employed throughout the paper. (B) Same as A, but for a genome-wide tiling of panTro5.}
\label{fig:figS11}
\end{figure}

\begin{figure}[!htb]
\centering
\includegraphics[width=4.5in]{img/figS12.1.pdf}
\caption[Higher-order chromosomal structure in humans and chimpanzees with alternative analysis choices.]{\textbf{S12. Higher-order chromosomal structure in humans and chimpanzees with alternative analysis choices.} (A) Across different resolutions (x-axis), we plotted the number of shared and species-specific domains (y-axis) identified with Arrowhead \cite{Durand.2016} on Juicer VC-normalized Hi-C maps from each individual. We called domain conservation here based on the method of Rao et al. \cite{Rao.2014} (highly similar results were observed with our 90\% reciprocal overlap method, described in the text and available in the github repository associated with the paper)...(continued on next page)}
\label{fig:figS12}
\end{figure}

\begin{figure}[!htb]
\centering
\includegraphics[width=4.5in]{img/figS12.2.pdf}
\contcaption{(continued from previous page) Domain count values represent the average interspecies sharing across all individuals, with no filtering for domain robustness (that is, assessing all domains discovered and orthologously mappable). Under this analysis paradigm we observe relatively low sharing across species ($\sim$60\% at 10kb). (B) Same as A, but this time, only considering TADs that were found across all 4 individuals within either one of the species (fixed TADs). Restricting to this subset increases the percentage of conservation to 78\%, although the set of TADs being examined is much smaller. (C) Same as A, but for boundaries instead of domains. Boundaries were defined as 15kb flanking regions at the edges of inferred Arrowhead domains. Because the TADs called by Arrowhead are nested, we merged boundaries here to obtain unique genomic intervals, rather than counting boundaries repeatedly. We then considered boundaries shared between individuals if they had any overlap. (D) Same as B, but for boundaries instead of domains (i.e. considering only boundaries fixed within species). Here, the highest estimate of conservation we obtain is 83\% of boundaries conserved across species at 10kb resolution. (E) Unsupervised hierarchical clustering of the pairwise proportions of shared TADs between all individuals in our study at a variety of resolutions, using the Rao et al. \cite{Rao.2014} methodology for calling conservation. The first letter in the labels demarcates the species (H for human and C for chimpanzee), and the following symbols indicate sex (male, M or female, F) and batch (1 or 2). Heatmaps are not necessarily symmetric because different numbers of TADs were discovered in different individuals; rows represent an individual's shared proportion of TADs (individual total) with each other individual. Highly similar clustering results were observed when using our domain conservation calling paradigm (shown in github repository associated with paper). (F) Same as E, but for boundaries instead of domains.}
\end{figure}

\begin{figure}[!htb]
\centering
\includegraphics[width=6in]{img/figS13.pdf}
\caption[Higher-order chromosomal structure in humans and chimpanzees with alternative analysis choices and genome builds.]{\textbf{S13. Higher-order chromosomal structure in humans and chimpanzees with alternative analysis choices and genome builds.} (A) Across different resolutions (x-axis), we plotted the number of shared and species-specific domains (y-axis) identified with Arrowhead \cite{Durand.2016} using the consensus map from each species. To call a domain as conserved here, we required that the Euclidean distance between the domain across species be less than the minimum of 50kb or 50\% the length of the TAD, based on the conservation calling method employed by Rao et al \cite{Rao.2014}. Results are highly similar to those seen in Fig \ref{fig:fig4}A. (B) Same as A, but for TAD boundaries instead of the domains themselves. Boundaries were defined as 15 kb flanking regions at the edges of inferred Arrowhead domains. In this case, conservation was called if there was any base pair overlap between boundaries. Unlike in Fig \ref{fig:fig4}B, boundaries were merged before calling conservation, in order to find unique boundary elements. This difference in analysis paradigms could have important consequences with a nested TAD caller such as Arrowhead \cite{Durand.2016}, but results are highly similar to those seen in Fig \ref{fig:fig4}B. (C) Same as A, but this time, performed on `high-density consensus' Hi-C maps that have been mapped to the hg38 and panTro6 genomes (rather than panTro5). Results are highly similar despite the improvement in genome quality build. (D) Same as B, but this time, on the hg38 and panTro6 genome assemblies.}
\label{fig:figS13}
\end{figure}

\begin{figure}[!htb]
\centering
\includegraphics[width=5in]{img/figS14.1.pdf}
\caption[Higher-order chromosomal structure in humans and chimpanzees with alternative algorithms (TopDom).]{\textbf{S14. Higher-order chromosomal structure in humans and chimpanzees with alternative algorithms (TopDom).} (A) Across different resolutions (x-axis), we plotted the number of shared and species-specific domains (y-axis) identified with TopDom \cite{Shin.2016} on HOMER-normalized Hi-C maps from each individual. We called domain conservation here based on the method of Rao et al. \cite{Rao.2014} (highly similar results were observed with our 90\% reciprocal overlap method, described in the text and available in the github repository associated with the paper)...(continued on next page)}
\label{fig:figS14}
\end{figure}

\begin{figure}[!htb]
\centering
\includegraphics[width=5in]{img/figS14.2.pdf}
\contcaption{(continued from previous page) Domain count values represent the average interspecies sharing across all individuals, with no filtering for domain robustness (that is, assessing all domains discovered and orthologously mappable). Under this analysis paradigm we observe relatively low sharing across species (maximum of 30\% at 25 kb). (B) Same as A, but this time, only considering TADs that were found across all 4 individuals within either one of the species (fixed TADs). Restricting to this subset increases the maximum percentage of conservation to 42\% at 25 kb resolution, although the set of TADs being examined is much smaller. (C) Same as A, but for TopDom \cite{Shin.2016} boundary inferences instead of domains. We considered boundaries shared between individuals if they had any overlap. (D) Same as B, but for boundaries instead of domains (i.e. considering only boundaries fixed within species). Here, the highest estimate of conservation we obtain is 76\% of boundaries conserved across species at 50 kb resolution. (E) Unsupervised hierarchical clustering of the pairwise proportions of shared TADs between all individuals in our study at a variety of resolutions, using the Rao et al. \cite{Rao.2014} methodology for calling conservation. The first letter in the labels demarcates the species (H for human and C for chimpanzee), and the following symbols indicate sex (male, M or female, F) and batch (1 or 2). Heatmaps are not necessarily symmetric because different numbers of TADs were discovered in different individuals; rows represent an individual's shared proportion of TADs (individual total) with each other individual. Highly similar clustering results were observed when using our domain conservation calling paradigm (shown in github repository associated with paper). (F) Same as E, but for boundaries instead of domains.}
\end{figure}

\begin{figure}[!htb]
\centering
\includegraphics[width=6in]{img/figS15.pdf}
\caption[Correlations between Hi-C and expression.]{\textbf{S15. Correlations between Hi-C and expression.} Density of Pearson correlations between RPKM expression values and log\textsubscript{2} HOMER-normalized contact frequencies across all 8 individuals. Solid lines indicate different sets of the observed data and dotted lines represent 10 permutations of the data. The Hi-C contact frequency chosen is that with the minimum FDR from linear modeling of contact frequency on species (see main text). The strong bimodal distribution of correlations between expression and contact suggests many instances where a contact difference between the species can lead to an increase (enhancer) or decrease (suppressor) of expression in the species where the contact is stronger.}
\label{fig:figS15}
\end{figure}

\begin{figure}[!htb]
\centering
\includegraphics[width=6in]{img/figS16.pdf}
\caption[Gene expression variance is explained by chromatin contacts for 8\% of DE genes.]{\textbf{S16. Gene expression variance is explained by chromatin contacts for 8\% of DE genes.} Plot of the species effect size in DE genes between models before (x-axis) and after (y-axis) conditioning on contact frequency. The Monte Carlo test of significance was used to construct the 95\% confidence interval and evaluate the significance of the indirect effect (species' effect on expression mediated through contact). Amongst DE genes, 8\% (116/1401) of genes showed a statistically significant effect of Hi-C contacts on expression levels (i.e. their 95\% confidence interval does not include zero).}
\label{fig:figS16}
\end{figure}

\begin{figure}[!htb]
\centering
\includegraphics[width=6in]{img/figS17.pdf}
\caption[Using a weighting scheme for chromHMM annotations increases the proportion of transcriptional and enhancer-like annotations.]{\textbf{S17. Using a weighting scheme for chromHMM annotations increases the proportion of transcriptional and enhancer-like annotations.} (A) Histogram showing the number of Hi-C loci (y-axis) assigned to each chromHMM annotation (x-axis) using maximum base pair overlap to assign each locus to a state. In the legend, `.' denotes that no annotation was found for a given bin. (TssA-Active TSS, TSSBiv-Bivalent/Poised TSS, BivFlnk-Flanking Bivalent TSS/Enh, EnhBiv-Bivalent Enhancer, ReprPC-Repressed PolyComb, ReprPCWk-Weak Repressed PolyComb, Quies-Quiescent/Low, TssAFlnk-Flanking Active TSS, TxFlnk-Transcription at gene 5' and 3', Tx-Strong transcription, TxWk-Weak transcription, EnhG-Genic Enhancers, Enh-Enhancers, ZNF/Rpts-ZNF genes and repeats, Het-Heterochromatin). (B) Same as A, only here, we assigned annotations after weighting chromHMM elements' overlaps with Hi-C loci by the reciprocal of their mean overlap in all our loci. This approach increases the number of 10kb Hi-C bins that are assigned to chromHMM annotations associated with transcriptional and enhancer activity (i.e. TssA, TssBiv, TssAFlnk, EnhG, Enh).}
\label{fig:figS17}
\end{figure}

\begin{figure}[!htb]
\centering
\includegraphics[width=6in]{img/figS18.pdf}
\caption[Overlap of epigenetic signatures and Hi-C contacts.]{\textbf{S18. Overlap of epigenetic signatures and Hi-C contacts.} (A) Density distribution of the base pair overlap between DHS peaks downloaded from ENCODE and our Hi-C loci. Plot is split between Hi-C loci that contact a promoter and those that do not. Inlay is a violin plot of the same distributions, with lines and numbers indicating pairwise t-tests of the mean, and their corresponding significance levels. (B) Density plot similar to A, but only considering Hi-C loci involved in contact with a promoter, and separating contacts into 4 classes, indicated by color: those that show differential contact between species, those that show differential expression between species, those that show both, and those that show neither. We used pairwise t-tests to compare differences in the mean overlap among the four classes of Hi-C loci. (C) Same as in B, but for the active histone mark H3K4me1. (D) Same as in B and C, but for the repressive histone mark H3K27me3.}
\label{fig:figS18}
\end{figure}

\begin{figure}[!htb]
\centering
\includegraphics[width=6in]{img/figS19.pdf}
\caption[Dynamics of chromHMM state among significant Hi-C contacts overlapping DE or non-DE genes.]{\textbf{S19. Dynamics of chromHMM state among significant Hi-C contacts overlapping DE or non-DE genes.} (A) Hi-C loci that make contact with promoters of genes that are not differentially expressed (DE) across species are ranked in order of decreasing DC FDR (x-axis). The y-axis shows cumulative proportion of chromHMM annotation assignments for all Hi-C loci at the given FDR or lower. (TssA-Active TSS, TSSBiv-Bivalent/Poised TSS, BivFlnk-Flanking Bivalent TSS/Enh, EnhBiv-Bivalent Enhancer, ReprPC-Repressed PolyComb, ReprPCWk-Weak Repressed PolyComb, Quies-Quiescent/Low, TssAFlnk-Flanking Active TSS, TxFlnk-Transcription at gene 5' and 3', Tx-Strong transcription, TxWk-Weak transcription, EnhG-Genic Enhancers, Enh-Enhancers, ZNF/Rpts-ZNF genes and repeats, Het-Heterochromatin). (B) Same as A, but only considering Hi-C loci making contact with promoters of genes that are differentially expressed (DE).}
\label{fig:figS19}
\end{figure}

\begin{figure}[!htb]
\centering
\includegraphics[width=5in]{img/figS20.pdf}
\caption[Reciprocal enrichments of differential expression and differential contact.]{\textbf{S20. Reciprocal enrichments of differential expression and differential contact.} (A) Enrichment of inter-species differentially contacting (DC) loci in genes with corresponding differences in expression (DE) between the species. The proportion of DE genes that are significantly DC (y-axis) is shown across a range of DE FDRs (x-axis). Colors indicate different DC FDR thresholds, and dashed lines indicate the proportion of DC loci expected by chance alone. (B) P values of Chi-squared tests of the null that there is no difference in proportion of DC loci among DE genes (y-axis), shown for a range of DE FDRs (x-axis). In both panels, the DE genes overlapping Hi-C loci were chosen to have the minimum FDR supporting inter-species difference in expression. (C) Similar to Fig \ref{fig:fig6}A, but using a weighted p-value combination technique \cite{Whitlock.2005} to integrate DC FDR across regions, instead of using the minimum FDR DC region. Once again, we observe enrichment of inter-species differentially expressed (DE) genes with corresponding differences in Hi-C contact frequencies (DC) between the species. The proportion of DC genes that are significantly DE (y-axis) is shown across a range of DC FDRs (x-axis). Colors indicate different DE FDR thresholds, and dashed lines indicate the proportion of DE genes expected by chance alone. (D) P values of Chi-squared tests of the null that there is no difference in proportion of DE genes among DC genes (y-axis), shown for a range of DC FDRs (x-axis).}
\label{fig:figS20}
\end{figure}

\pagebreak
\clearpage

\section{Supplemental Tables}

\begin{table}[!htb]
\captionstyle{\raggedright}
\caption[S1. HOMER-called contacts, H21792.]{\textbf{S1. HOMER-called contacts, H21792.}}
\label{tab:s1}
\end{table}

\begin{table}[!htb]
\captionstyle{\raggedright}
\caption[S2. HOMER-called contacts, H28126.]{\textbf{S2. HOMER-called contacts, H28126.}}
\label{tab:s2}
\end{table}

\begin{table}[!htb]
\captionstyle{\raggedright}
\caption[S3. HOMER-called contacts, H28815.]{\textbf{S3. HOMER-called contacts, H28815.}}
\label{tab:s3}
\end{table}

\begin{table}[!htb]
\captionstyle{\raggedright}
\caption[S4. HOMER-called contacts, H28834.]{\textbf{S4. HOMER-called contacts, H28834.}}
\label{tab:s4}
\end{table}

\begin{table}[!htb]
\captionstyle{\raggedright}
\caption[S5. HOMER-called contacts, C3649.]{\textbf{S5. HOMER-called contacts, C3649.}}
\label{tab:s5}
\end{table}

\begin{table}[!htb]
\captionstyle{\raggedright}
\caption[S6. HOMER-called contacts, C40300.]{\textbf{S6. HOMER-called contacts, C40300.}}
\label{tab:s6}
\end{table}

\begin{table}[!htb]
\captionstyle{\raggedright}
\caption[S7. HOMER-called contacts, C3624.]{\textbf{S7. HOMER-called contacts, C3624.}}
\label{tab:s7}
\end{table}

\begin{table}[!htb]
\captionstyle{\raggedright}
\caption[S8. HOMER-called contacts, C3651.]{\textbf{S8. HOMER-called contacts, C3651.}}
\label{tab:s8}
\end{table}

\begin{table}[!htb]
\captionstyle{\raggedright}
\caption[S9. Orthology calling statistics.]{\textbf{S9. Orthology calling statistics.}}
\label{tab:s9}
\end{table}

\begin{table}[!htb]
\captionstyle{\raggedright}
\caption[S10. Differentially contacting (DC) regions.]{\textbf{S10. Differentially contacting (DC) regions.}}
\label{tab:s10}
\end{table}

\begin{table}[!htb]
\captionstyle{\raggedright}
\caption[S11. Human consensus Arrowhead-inferred TADs.]{\textbf{S11. Human consensus Arrowhead-inferred TADs.}}
\label{tab:s11}
\end{table}

\begin{table}[!htb]
\captionstyle{\raggedright}
\caption[S12. Chimpanzee consensus Arrowhead-inferred TADs.]{\textbf{S12. Chimpanzee consensus Arrowhead-inferred TADs.}}
\label{tab:s12}
\end{table}

\begin{table}[!htb]
\captionstyle{\raggedright}
\caption[S13. Human-Chimpanzee orthologous TAD coordinates.]{\textbf{S13. Human-Chimpanzee orthologous TAD coordinates.}}
\label{tab:s13}
\end{table}

\begin{table}[!htb]
\captionstyle{\raggedright}
\caption[S14. ChromHMM genic enhancer annotation enrichments.]{\textbf{S14. ChromHMM genic enhancer annotation enrichments.}}
\label{tab:s14}
\end{table}
\break

\begin{table}[!htb]
\captionstyle{\raggedright}
\caption[S15. ENCODE chromatin mark sources.]{\textbf{S15. ENCODE chromatin mark sources.}}
\label{tab:s15}
\end{table}

\begin{table}[!htb]
\captionstyle{\raggedright}
\caption[S16. Sample metadata.]{\textbf{S16. Sample metadata.}}
\label{tab:s16}
\end{table}

\begin{table}[!htb]
\captionstyle{\raggedright}
\caption[S17. 10kb Arrowhead TAD inferences, H21792.]{\textbf{S17. 10kb Arrowhead TAD inferences, H21792.}}
\label{tab:s17}
\end{table}

\begin{table}[!htb]
\captionstyle{\raggedright}
\caption[S18. 10kb Arrowhead TAD inferences, H28126.]{\textbf{S18. 10kb Arrowhead TAD inferences, H28126.}}
\label{tab:s18}
\end{table}

\begin{table}[!htb]
\captionstyle{\raggedright}
\caption[S19. 10kb Arrowhead TAD inferences, H28815.]{\textbf{S19. 10kb Arrowhead TAD inferences, H28815.}}
\label{tab:s19}
\end{table}

\begin{table}[!htb]
\captionstyle{\raggedright}
\caption[S20. 10kb Arrowhead TAD inferences, H28834.]{\textbf{S20. 10kb Arrowhead TAD inferences, H28834.}}
\label{tab:s20}
\end{table}

\begin{table}[!htb]
\captionstyle{\raggedright}
\caption[S21. 10kb Arrowhead TAD inferences, C3649.]{\textbf{S21. 10kb Arrowhead TAD inferences, C3649.}}
\label{tab:s21}
\end{table}

\begin{table}[!htb]
\captionstyle{\raggedright}
\caption[S22. 10kb Arrowhead TAD inferences, C40300.]{\textbf{S22. 10kb Arrowhead TAD inferences, C40300.}}
\label{tab:s22}
\end{table}

\begin{table}[!htb]
\captionstyle{\raggedright}
\caption[S23. 10kb Arrowhead TAD inferences, C3624.]{\textbf{S23. 10kb Arrowhead TAD inferences, C3624.}}
\label{tab:s23}
\end{table}

\begin{table}[!htb]
\captionstyle{\raggedright}
\caption[S24. 10kb Arrowhead TAD inferences, C3651.]{\textbf{S24. 10kb Arrowhead TAD inferences, C3651.}}
\label{tab:s24}
\end{table}

