\chapter{Conclusion}\label{conclusion}

\section{Evolutionary and Gene Regulatory Implications of this Work}
The field of human genetics has already accomplished much in connecting genetic variation to complex trait and disease variation between individuals.Outstanding challenges remain in characterizing the mechanisms of action for all these variants, and understanding the relative contributions of these different mechanisms to speciation, adaptation, and inter-individual trait variation. Although our understanding is rapidly expanding, we are still far from obtaining a comprehensive picture of gene regulation. The findings detailed herein corroborate the idea that assessing 3D genome structure is a crucial next piece of the puzzle. In Chapter 2, collaborators and I measured 3D genome structure and gene expression across human and chimpanzee induced pluripotent stem cells (iPSCs), revealing that differences in 3D genome structure may contribute to differential gene expression across these species. We found that, at the lowest scale (i.e. individual gene regulatory DNA loops), human and chimpanzee chromatin contacts are fairly conserved. This would imply that, at least in iPSCs, individual gene-cis-regulatory element (CRE) interactions do not vary significantly between humans and chimpanzees. However, this does not necessarily mean that the origins of regulatory novelty must lie elsewhere. Chromatin state in iPSCs is generally more open and "permissive" than in differentiated cell types \cite{Spivakov and Fisher 2007 (Epigenetic signatures of stem-cell identity)}, and thus we may expect lower divergence in gene-CRE loops across species in iPSCs than in other cell types. Interestingly, we also observed evidence for large sets of species-biased differences in loop strength on individual chromosomes that have experienced large-scale rearrangements between humans and chimpanzees. This suggests that such rearrangements may help drive interspecies regulatory novelty in 3D chromatin interactions, although more functional follow-up will be required to confirm this notion, given conflicting results from some other comparative studies \cite{Lazar et al. 2018 (Epigenetic maintenance of topological domains in the highly rearranged gibbon genome), Krefting et al. 2018 (Evolutionary stability of topologically associating domains is associated with conserved gene regulation)}.

Regardless of the precise level of conservation/divergence in DNA looping, we found ample evidence for pairs of loci exhibiting differential contact (DC) across species. When we overlapped these data with RNA-seq data assessing gene expression levels, we observed strong correlations between interspecies contact and expression differences for differentially expressed (DE) genes overlapping our Hi-C loci. The fact that we did not observe similarly strong correlations for non-DE genes implies that variation in chromatin contacts plays a role in DE. Further corroborating this notion, we observed a significant enrichment for DE amongst genes we classified as DC across species--and vice versa. As previously stated, the observational nature of the study meant we could not directly infer a causal relationship between DC and DE. However, our mediation analysis found that up to 8\% of DE genes may have a significant portion of their expression variation explained by variation in chromatin contacts. Placed in the context of other studies that observed perturbations in chromatin contact affecting gene expression \cite{Lupianez et al. 2015 (Disruptions of topological chromatin domains cause...), Siersbæk et al. 2017 (Dynamic rewiring of promoter-anchored chromatin loops during adipocyte differentiation)}, our findings suggest that species-specific differences in 3D genomic contacts are indeed a driver of species-specific expression. This conclusion is also supported by our observation that, compared to contacts not involving a promoter, promoter-associated contacts are enriched for more active chromHMM state assignments. The intuitive conclusion from this result is that loci making contact with a promoter are likely involved in active regulation of the corresponding gene. Similarly, we observed that joint DE/DC loci identified in our study are enriched for a wide variety of functional epigenetic marks as compared to non-DE/DC loci. Under the common paradigm that most chromatin is not accessible and thus not active in any given cell type \cite{Thurman et al. 2012 (The accessible chromatin landscape of the human genome)}, these functional annotation enrichments suggest that the identified joint DE/DC loci represent functionally relevant stretches of DNA between the species. Based on all these results, it may be tempting to speculate that 3D genome conformation is one of the most basal elements laying the groundwork for a broad cascade of events dictating gene regulation. Unfortunately, this conclusion seems somewhat premature absent a bevy of mechanistic perturbation studies, and given more recent conflicting results about the order and nature of events with respect to genome conformation and observed differences in gene expression \cite{Jiang et al. 2020 (Genome-wide analyses of chromatin interactions after the loss of Pol I, Pol II, and Pol III), Ghavi-Helm et al. 2019 (Highly rearranged chromosomes reveal uncoupling between genome topology and gene expression), Espinola et al. 2020 (biorxiv, Cis-regulatory chromatin loops arise before TADs and gene activation, and are independent of cell fate during development), Ing-Simmons 2020 (biorxiv, Independence of 3D chromatin conformation and gene regulation during Drosophila dorsoventral patterning), Alexander et al. 2019 (Live-cell imaging reveals enhancer-dependent Sox2 transcription in the absence of enhancer proximity), Benabdallah et al. 2019 (Decreased Enhancer-promoter proximity accompanying enhancer activation)}.

Beyond exploring regulatory loop conservation and its effects on expression, the data collected in Chapter 2 also enabled us to examine higher-order chromatin structure, such as topologically associating domains (TADs), across the species. We found relatively weak conservation of TAD structures as compared to regulatory loops and other epigenetic phenotypes previously compared between humans and chimpanzees. While this might point to TAD variation as a significant source of regulatory novelty, we were unable to find concrete examples of interspecies TAD differences affecting differential expression. This does not, however, preclude a significant role for TAD variation in speciation and interspecies expression divergence; as I discuss further below, TADs are poorly defined and difficult to robustly infer \cite{Dali and Blanchette 2017}. Regardless, the observed low interspecies TAD conservation was surprising, given the prevailing notion in the field that TADs are highly conserved across species. In large part, this incongruence motivated our critical assessment of the evidence for evolutionary TAD conservation, detailed in Chapter 3. A thorough review of the available data suggest that, while there is certainly some evidence for TAD conservation across mammalian species, it is not compelling enough to claim TADs are highly conserved. Something about why this matters WRT evolution and new paradigms of gene regulation (origins of regulatory novelty).

On a broad scale, my thesis work has confirmed the idea that 3D genome organization is an integral feature affecting the evolution of gene regulation. It is important to understand the limitations of this work, and consequent avenues for future research.

\section{Challenges and next steps}
The true extent of interspecies divergence in gene-CRE interactions may be higher than our estimate, given the underpowered nature of the Hi-C assay \cite{Belton et al. 2012 (Hi-C: A comprehensive technique to capture the conformation of genomes)} and our limited number of individuals from each species. Still, our analysis was carried out in a robust quantitative fashion that is likely to give more accurate estimates of inter-species conservation than simplistic traditional approaches (i.e. assessing conservation via a venn diagram of overlap in significant chromatin contacts per-species). Directly testing each contact identified as significant in any individual for inter-species differences allowed us to largely sidestep the issue of incomplete power, avoiding an inflated estimate of divergence. Future work assaying 3D genome structure at high sequencing depth across a variety of cell types will be needed to robustly assess chromatin loop divergence amongst primate species.

The precise functional significance and evolutionary impact of these loci could be more thoroughly assessed and polarized in the future by adding chimpanzee epigenetic mark data (we only used publicly-available human data). Such an analysis would be particularly interesting for assessing which DC loci are most likely to have species-specific effects on gene regulation. Undergone differential CRE evolution between species. been functionally altered at multiple levels through the course of human and chimpanzee evolution.
In our case, we were only able to overlap epigenetic mark data from humans, but future studies integrating such data across species


Gene expression affected by chromatin contacts--if we had collected RNA-seq concomitantly (rather than previously from same cell lines), if we had sequenced deeper to go lower than 10kb resolution

TADs affecting gene expression and being conserved across cells and species


\section{The state of the 3D genome field, and future perspectives}
Why we need better single-cell methods
Why we need better methods to assess 3D structure from a biochemical perspective--non-proximity-ligation based
Why we need to move towards larger panels of individuals and connect genetic variation to 3D genome variation \cite{Gorkin et al. 2019 (Common DNA sequence variation influences 3-dimensional conformation of the human genome)}
Why we need more robust methods to compare 3D genome structure across species \cite{Yang et al 2019 (Comparing 3D genome organization in multiple species using phylo-HMRF)}
	-Methods exist to compare 3D genome structure across conditions (diffHiC), we need more support for evolutionary comparisons.