\chapter{Conclusion}\label{conclusion}

\section{Evolutionary and gene regulatory implications of this work}
The field of human genetics has already accomplished much in connecting genetic variation to complex trait and disease variation between individuals.Outstanding challenges remain in characterizing the mechanisms of action for all these variants, and understanding the relative contributions of these different mechanisms to speciation, adaptation, and inter-individual trait variation. Although our understanding is rapidly expanding, we are still far from obtaining a comprehensive picture of gene regulation. The findings detailed herein corroborate the idea that assessing 3D genome structure is a crucial next piece of the puzzle. In Chapter 2, collaborators and I measured 3D genome structure and gene expression across human and chimpanzee induced pluripotent stem cells (iPSCs), revealing that differences in 3D genome structure may contribute to differential gene expression across these species. We found that, at the lowest scale (i.e. individual gene regulatory DNA loops), human and chimpanzee chromatin contacts are fairly conserved. This would imply that, at least in iPSCs, individual gene-cis-regulatory element (CRE) interactions do not vary significantly between humans and chimpanzees. However, this does not necessarily mean that the origins of regulatory novelty must lie elsewhere. Chromatin state in iPSCs is generally more open and "permissive" than in differentiated cell types \cite{Spivakov and Fisher 2007 (Epigenetic signatures of stem-cell identity)}, and thus we may expect lower divergence in gene-CRE loops across species in iPSCs than in other cell types. Interestingly, we also observed evidence for large sets of species-biased differences in loop strength on individual chromosomes that have experienced large-scale rearrangements between humans and chimpanzees. This suggests that such rearrangements may help drive interspecies regulatory novelty in 3D chromatin interactions, although more functional follow-up will be required to confirm this notion, given conflicting results from some other comparative studies \cite{Lazar et al. 2018 (Epigenetic maintenance of topological domains in the highly rearranged gibbon genome), Krefting et al. 2018 (Evolutionary stability of topologically associating domains is associated with conserved gene regulation)}.

Regardless of the precise level of conservation/divergence in DNA looping, we found ample evidence for pairs of loci exhibiting differential contact (DC) across species. When we overlapped these data with RNA-seq data assessing gene expression levels, we observed strong correlations between interspecies contact and expression differences for differentially expressed (DE) genes overlapping our Hi-C loci. The fact that we did not observe similarly strong correlations for non-DE genes implies that variation in chromatin contacts plays a role in DE. Further corroborating this notion, we observed a significant enrichment for DE amongst genes we classified as DC across species--and vice versa. As previously stated, the observational nature of the study meant we could not directly infer a causal relationship between DC and DE. However, our mediation analysis found that up to 8\% of DE genes may have a significant portion of their expression variation explained by variation in chromatin contacts. Placed in the context of other studies that observed perturbations in chromatin contact affecting gene expression \cite{Lupianez et al. 2015 (Disruptions of topological chromatin domains cause...), Siersbæk et al. 2017 (Dynamic rewiring of promoter-anchored chromatin loops during adipocyte differentiation)}, our findings suggest that species-specific differences in 3D genomic contacts are indeed a driver of species-specific expression. This conclusion is also supported by our observation that, compared to contacts not involving a promoter, promoter-associated contacts are enriched for more active chromHMM state assignments. The intuitive conclusion from this result is that loci making contact with a promoter are likely involved in active regulation of the corresponding gene. Similarly, we observed that joint DE/DC loci identified in our study are enriched for a wide variety of functional epigenetic marks as compared to non-DE/DC loci. Under the common paradigm that most chromatin is not accessible and thus not active in any given cell type \cite{Thurman et al. 2012 (The accessible chromatin landscape of the human genome)}, these functional annotation enrichments suggest that the identified joint DE/DC loci represent functionally relevant stretches of DNA between the species. Based on all these results, it may be tempting to speculate that 3D genome conformation is one of the most basal elements laying the groundwork for a broad cascade of events dictating gene regulation. Unfortunately, this conclusion seems somewhat premature absent a bevy of mechanistic perturbation studies, and given more recent conflicting results about the order and nature of events with respect to genome conformation and observed differences in gene expression \cite{Jiang et al. 2020 (Genome-wide analyses of chromatin interactions after the loss of Pol I, Pol II, and Pol III), Ghavi-Helm et al. 2019 (Highly rearranged chromosomes reveal uncoupling between genome topology and gene expression), Espinola et al. 2020 (biorxiv, Cis-regulatory chromatin loops arise before TADs and gene activation, and are independent of cell fate during development), Ing-Simmons 2020 (biorxiv, Independence of 3D chromatin conformation and gene regulation during Drosophila dorsoventral patterning), Alexander et al. 2019 (Live-cell imaging reveals enhancer-dependent Sox2 transcription in the absence of enhancer proximity), Benabdallah et al. 2019 (Decreased Enhancer-promoter proximity accompanying enhancer activation)}. Although cause and consequence are still difficult to disentangle in this framework, there is no doubt that 3D genome conformation is an important facet affecting the evolution of gene regulation. 

Beyond exploring regulatory loop conservation and its effects on expression, the data collected in Chapter 2 also enabled us to examine higher-order chromatin structure, such as topologically associating domains (TADs), across the species. We found relatively weak conservation of TAD structures as compared to regulatory loops and other epigenetic phenotypes previously compared between humans and chimpanzees. While this might point to TAD variation as a significant source of regulatory novelty, we were unable to find concrete examples of interspecies TAD differences affecting differential expression. This does not, however, preclude a significant role for TAD variation in speciation and interspecies expression divergence; as I discuss further below, this lack of signal could be due to TADs being poorly defined and difficult to robustly infer \cite{Dali and Blanchette 2017}. Regardless, the observed low interspecies TAD conservation was surprising, given the prevailing notion in the field that TADs are highly conserved across species \cite{Dixon et al. 2012, Rao et al. 2014}. In large part, this incongruence motivated our critical assessment of the evidence for evolutionary TAD conservation, detailed in Chapter 3. A thorough review of the available data suggest that, while there is certainly some evidence for TAD conservation across mammalian species, it is not compelling enough to claim TADs are highly conserved. The validity of this notion is important to consider because, if true, it implies that TAD variation does not play a significant role in speciation. As addressed further in the final section of this chapter, analytical and definitional issues stymie a robust assessment of interspecies TAD variation and preclude a thorough understanding of TADs' impact on gene expression. This much is evident from the fact that, although the TAD inference algorithms we employed found numerous differences between the species, visualizations of the corresponding contact maps did not appear significantly different between humans and chimpanzees. Thus, the results of our own TAD comparisons do not necessarily conflict with previous findings. It is possible that differences in TADs play a significant role in differences observed between primate species, but it is difficult to support or refute this notion with any confidence, given the current state of the field. Despite this, my thesis work has broadly confirmed the idea that 3D genome organization is an integral feature affecting the evolution of gene regulation. At the same time, it is important to understand the limitations of this work, and consequently, avenues for future research.

\section{Limitations and next steps}
There are a number of limitations to this research program that should be considered to inform avenues for future research. For one, the true extent of interspecies divergence in gene-CRE interactions may be higher than our estimate, given the underpowered nature of the Hi-C assay \cite{Belton et al. 2012 (Hi-C: A comprehensive technique to capture the conformation of genomes)} and our limited number of individuals from each species. Still, our analysis was carried out in a robust quantitative fashion that is likely to give more accurate estimates of inter-species conservation than simplistic approaches other studies have used (i.e. assessing conservation via a venn diagram of overlap in significant chromatin contacts per-species) \cite{Dixon 2012, Rao 2014}. Directly testing each contact identified as significant in any individual for inter-species differences allowed us to largely sidestep the issue of incomplete power, avoiding an overinflated estimate of divergence. While numerous methods have been proposed to quantitatively compare regulatory loop strength across different biological conditions \cite{Lun and Smyth 2015 (DiffHiC), Paulsen et al. 2014 (HiBrowse: multi-purpose statistical analysis of genome-wide chromatin 3D organization), Djekidel et al. 2018 (FIND: difFerential chromatin INteractions Detection using a spatial poisson process), Fernandez et al. 2020 (3DeFDR: statistical methods for identifying cell type-specific looping interactions in 5C and Hi-C data)}, sparse novel techniques have only very recently emerged for running similar comparisons across species \cite{Yang et al. 2019 (Comparing 3D genome organization in multiple species using phylo-hmrf), Nuriddinov and Fishman 2019 (C-intersecture--a computational tool for interspecies comparison of genome architecture)}. When we began analyzing the data collected in Chapter 2, these techniques had not yet been published, but applying them to these and similar data in the future would be of great interest for robustly assessing primate divergence in regulatory chromatin looping. In a similar vein, the sequencing depth in our own study allowed for assessment of chromatin contacts at a 10kb resolution, but a comprehensive picture of inter-primate differences in chromatin loops will require deeper sequencing to enable sub-kilobase resolution and analysis of finer-scale loops. Lastly and as noted above, our comparisons were only performed in iPSCs, which tend to have more permissive regulatory landscapes than differentiated cell types \cite{Spivakov and Fisher 2007 (Epigenetic signatures of stem-cell identity)}. Comparison of 3D chromatin structure across species in other cell types would thus be highly desirable, and may reveal greater divergence in gene-CRE loops than that observed in our own work. Ideally, this would be performed on isogenic samples to reduce the confounding effects of genetic variation, which could be accomplished by differentiating the same iPSC lines into a variety of terminal cell types.

Another important limitation to consider is that, when examining functional enrichments, we only overlapped our DE/DC loci with publicly-available epigenetic data from humans. The precise functional significance and evolutionary impact of these loci could be more thoroughly assessed and polarized in the future by adding chimpanzee epigenetic mark data. Such an analysis would be particularly interesting for assessing which DC loci have undergone differential CRE evolution between species, and thus are more likely to have species-specific effects on gene regulation. The mediation analysis we utilized to assess the impact of DC on DE could be expanded to include epigenetic mark data across species, improving power to predict gene expression differences \cite{Karli? et al. 2010 (Histone modification levels are predictive for gene expression)}. More broadly, our analyses integrating Hi-C data and RNA-seq data could be improved in a number of ways that may find more expression variation explained by chromatin contact variation. As is also the case for assessment of conservation, deeper sequencing could provide better resolution of individual gene-CRE interactions, making it easier to tease out the effects of chromatin contact on expression. At the 10kb resolution we used, there were instances of multiple genes being assigned to the same Hi-C bin, likely obscuring interesting signals that could be observed in finer-scale data. Similarly, greater signals of association between chromatin contact and expression might have been observed if the RNA-seq and Hi-C data were collected concomitantly. Although our RNA-seq data came from the same cell lines, they were collected previously by different researchers culturing the cells in slightly different conditions. Concomitant collection would be more likely to maintain (and thus detect) weaker links between chromatin conformation and gene expression that may have been concealed in our own data. In some sense, our observational collection of these data across species represents an experimental paradigm for "natural perturbation" of chromatin structure and gene expression. At the same time, a thorough understanding of DC affecting DE would require more precise targeted perturbations, altering regulatory loop strength in one species and expecting to see corresponding expression "rescue" to comparable levels observed in the other species.

Our inferences regarding conservation of TAD structure could also be improved upon with functional and perturbational follow-up studies. As mentioned in the previous section, our algorithmic inference of TAD divergence often appeared fairly conserved upon visual inspection. This is probably largely due to issues with TAD identification (discussed further below), but could be mitigated with further functional characterizations. In particular, divergence in TAD structure could be more confidently characterized by also collecting ChIP-seq for CCCTC-binding factor (CTCF), a protein centrally involved in anchoring chromatin loops and demarcating TAD boundaries \cite{Rao et al. 2014, Zuin et al. 2014 (Cohesin and CTCF differentially affect chromatin architecture and gene expression in human cells), Phillips-Cremins et al. 2013 (Architectural protein subclasses shape 3D organization of genomes during lineage commitment), Dixon et al. 2012, Nora et al. 2012 (Spatial partitioning of the regulatory landscape of the X-inactivation centre)}. Overlaying these data with Hi-C data could help differentiate between instances of TAD divergence that are reflective of true biology (i.e. their boundaries show differences in CTCF binding across species), and divergence that is driven by technical issues (i.e. minimal difference in CTCF binding, but the inference algorithm fails to detect a TAD in one species where it nonetheless appears present). Situations falling into the former category could be further validated and characterized via CRISPR-based perturbations to the differential CTCF binding site in both species. Specifically, one could abrogate the binding site in the species where CTCF is strongly bound, verify reduced binding via ChIP-seq, and observe subsequent effects on TAD structure and corresponding gene expression. Carrying out the reciprocal experiment (i.e. creating a binding site in the species where CTCF is not bound) would also be useful. In both instances, these perturbations should drive TAD structure and local gene expression to appear more similar to the species where CTCF binding was not altered. If this expectation ends up being incorrect, that is in and of itself an extremely interesting result, and could spur further research into the mechanisms driving TAD formation and divergence across species. Furthermore, integrating the other data types collected with CTCF occupancy differences across species would add another intriguing layer of regulation that could help explain individual instances of DC and DE, and their effect on one another. Lastly, although it has been examined in some previous studies within a single species \cite{Sauerwald et al. 2019 (Analysis of the structural variability of topologically associated domains as revealed by Hi-C), Sauerwald and Kingsford 2018 (Quantifying the similarity of topological domains across normal and cancer human cell types), Schmitt et al. 2016 (A compendium of chromatin contact maps reveals spatially active regions in the human genome), Dixon et al. 2015 (Chromatin architecture reorganization during stem cell differentiation)}, estimating TAD sharing between tissues and cell types across primate species would be of great interest for elucidating the role these structures may play in differentiation and development, and how this may differ across evolution. Unfortunately, as I discuss in Chapter 3 and elaborate upon below, robust inferences regarding TAD conservation, function, and tissue-specificity are still severely hampered by imprecise and variable TAD definitions.

\section{The state of the 3D genome field, challenges, and future perspectives}
3D genomics is an exciting and relatively young field of epigenetic research. While the novelty of the field allows for high-impact discoveries and inferences, it also presents unique challenges. Hi-C is regarded as a revolutionary technology enabling genome-wide assessment of 3D genomic contacts, but it is also known to have a poor signal:noise ratio \cite{Lajoie et al. 2016 (The Hitchhiker's Guide to Hi-C Analysis: Practical guidelines), Yardimci et al. 2019 (Measuring the reproducibility and quality of Hi-C data)}. Several factors contribute to this issue. The method creates chimeric molecules by ligating physically proximal restriction fragments together, and the number of possible pairwise fragment interactions is very high, regardless of the restriction enzyme used. This means that, in order to achieve statistical power to detect significant contacts, reads from Hi-C data must be binned into fixed-size intervals tiling the genome \cite{Pal et al. 2018 (Hi-C analysis: from data generation to integration)}. In addition to the binning problem, the fact that Hi-C is a proximity-ligation based method also introduces spurious ligation products.
Why we need better single-cell methods \cite{Zhou et al. 2019 (Robust single-cell Hi-C clustering by convolution- and random-walk-based imputation), Ramani et al. 2019 (Sci-Hi-C: A single-cell Hi-C method for mapping 3D genome organization in large number of single cells), Nagano et al. 2013 (Single-cell Hi-C reveals cell-to-cell variability in chromosome structure)}
How transitioning from dilution to in-situ Hi-C reduced noise \cite{Nagano et al. 2015}
Why we need to arrive on a robust method to analyze the data \cite{Forcato et al. 2017 (Comparison of computational methods for Hi-C data analysis), Ay and Noble 2015 (Analysis methods for studying the 3D architecture of the genome)}
Why we need better methods to assess 3D structure from a biochemical perspective--non-proximity-ligation based \cite{You et al. 2020 (Direct DNA crosslinking with CAP-C uncovers transcription-dependent chromatin organization at high resolution)}
Why we need to move towards larger panels of individuals and connect genetic variation to 3D genome variation \cite{Gorkin et al. 2019 (Common DNA sequence variation influences 3-dimensional conformation of the human genome)}
Why we need more robust methods to compare 3D genome structure across species \cite{Yang et al 2019 (Comparing 3D genome organization in multiple species using phylo-HMRF)}
	-Methods exist to compare 3D genome structure across conditions (diffHiC), we need more support for evolutionary comparisons.